<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Molecular Cloning | Protocol 4 Print Version</TITLE>
<META http-equiv=Content-Type content="text/html; charset=windows-1251"><!--================== Designed and Developed by WriteDesign =================--><!--======================= Powered by RomeBlack, Inc. =======================--><!--=== Copyright 2000, 2001 Cold Spring Harbor Labs. All Rights Reserved. ===-->
<META content="" name=keywords>
<META 
content="Two oligonucleotide primers are hybridized to the same strand of a denatured double-stranded recombinant plasmid. One primer (the mutagenic primer) introduces the desired mutation into the target sequences, and the second primer carries a mutation that destroys a unique restriction site in the plasmid. Both primers are elongated in a reaction catalyzed by bacteriophage T4 or T7 DNA polymerase. Nicks in the strand of newly synthesized DNA are sealed with bacteriophage T4 DNA ligase. The product of the first part of the method is a heteroduplex plasmid consisting of a wild-type parental strand and a new full-length strand that carries the desired mutation but no longer contains the unique restriction site.Residual wild-type plasmid molecules are then linearized by digestion with the appropriate restriction enzyme.  The mixture of circular heteroduplex DNA and linear wild-type DNA is then used to transform a strain of E. coli that is deficient in repair of mismatched bases.  Because linear DNA transforms 10-1000-fold less efficiently than circular DNA, many of the wild-type molecules are unable to reestablish themselves in E. coli. The circular heteroduplex molecules, however, begin to replicate. Because the mismatched bases are not repaired, the first round of replication generates a wild-type plasmid that carries the original restriction site and a mutated plasmid that does not. DNA from the first set of transformants is recovered, digested once more with the same restriction enzyme to linearize the wild-type molecules, and then used to transform a standard laboratory strain of E. coli. This biochemical selection can be sufficiently powerful to ensure that a high proportion of the resulting transformants carry the desired mutation." 
name=description>
<META content="MSHTML 5.50.4134.600" name=GENERATOR></HEAD>
<BODY bgColor=#ffffff background=../izo/fon.gif>
<table width=600 border="0" cellspacing="0" cellpadding="0"><tr><td width=600><IMG height=49 hspace=4 src="../izo/logo_csh.gif" 
width=169 align=left vspace=4><IMG height=49 hspace=4 
src="../izo/print_header.gif" width=293 align=left vspace=4><BR 
clear=all><BR><FONT face="arial,verdana, helvetica" size=4><STRONG>Chapter 13, 
Protocol 4</STRONG></FONT><BR><BR clear=all><FONT face=arial,verdana,helvetica 
size=3><STRONG>Oligonucleotide-directed Mutagenesis by Elimination of a Unique 
Restriction Site (USE Mutagenesis)</STRONG></FONT><BR><BR><FONT 
face="arial,verdana, helvetica" size=2>Two oligonucleotide primers are 
hybridized to the same strand of a denatured double-stranded recombinant 
plasmid. One primer (the mutagenic primer) introduces the desired mutation into 
the target sequences, and the second primer carries a mutation that destroys a 
unique restriction site in the plasmid. Both primers are elongated in a reaction 
catalyzed by bacteriophage T4 or T7 DNA polymerase. Nicks in the strand of newly 
synthesized DNA are sealed with bacteriophage T4 DNA ligase. The product of the 
first part of the method is a heteroduplex plasmid consisting of a wild-type 
parental strand and a new full-length strand that carries the desired mutation 
but no longer contains the unique restriction site.<BR>Residual wild-type 
plasmid molecules are then linearized by digestion with the appropriate 
restriction enzyme. The mixture of circular heteroduplex DNA and linear 
wild-type DNA is then used to transform a strain of <I>E. coli</I> that is 
deficient in repair of mismatched bases. Because linear DNA transforms 
10-1000-fold less efficiently than circular DNA, many of the wild-type molecules 
are unable to reestablish themselves in <I>E. coli</I>. The circular 
heteroduplex molecules, however, begin to replicate. Because the mismatched 
bases are not repaired, the first round of replication generates a wild-type 
plasmid that carries the original restriction site and a mutated plasmid that 
does not. DNA from the first set of transformants is recovered, digested once 
more with the same restriction enzyme to linearize the wild-type molecules, and 
then used to transform a standard laboratory strain of <I>E. coli</I>. This 
biochemical selection can be sufficiently powerful to ensure that a high 
proportion of the resulting transformants carry the desired mutation.</FONT> 
<BR><BR>
<TABLE border=1>
  <TBODY>
  <TR>
    <TD width=700><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" 
      width=11 border=0><FONT face=arial,verdana,helvetica 
      size=2><STRONG>CAUTION</STRONG></FONT><BR><IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0><FONT 
      face="arial,verdana, helvetica" size=2> <STRONG>RECIPE</STRONG></FONT> 
  </TD></TR></TBODY></TABLE><BR><BR><FONT face=arial,verdana,helvetica 
size=3><STRONG>MATERIALS</STRONG></FONT><BR><BR>
<TABLE cellPadding=0 cellborder="0">
  <TBODY>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Buffers and Solutions</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>10x Annealing 
      buffer (13-4)</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>10x Synthesis 
      buffer</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Enzymes and Buffers</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Bacteriophage 
      T4 DNA ligase</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Bacteriophage 
      T4 DNA polymerase or Sequenase</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Unique site 
      restriction endonuclease</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Nucleic Acids and Oligonucleotides</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Mutagenic 
      primer</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Selection 
      primer</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Both 
      mutagenic and selection primers must anneal to the same strand of the 
      target DNA, and the 5&acute; end of each primer must be phosphorylated. The 
      oligonucleotide primers should be purified by FPLC or PAGE before use 
      (please see <A href="../10/01.htm">Chapter 
      10, Protocol 1</A> or <A href="../10/05.htm">Chapter 
      10, Protocol 5</A>).</I><BR><I>Several companies (CLONTECH, Pharmacia) 
      sell selection primers and some companies (e.g., Pharmacia) also market 
      pairs of “toggle” primers that are used for forward and reverse conversion 
      of restriction sites and allow sequential rounds of mutagenesis without 
      subcloning of the template.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Plasmid 
      DNA</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Closed 
      circular plasmid DNA, purified either by chromatography through a 
      commercial resin or by the alkaline lysis method (please see <A href="../01/02.htm">Chapter 
      1, Protocol 2</A> or <A href="../01/09.htm">Chapter 
      1, Protocol 9</A>).</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Media</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>LB agar plates 
      containing the appropriate antibiotic</FONT> <IMG height=12 alt="" 
      hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>LB medium 
      containing the appropriate antibiotic</FONT> <IMG height=12 alt="" 
      hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Additional Reagents</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Steps 7 and 14 
      of this protocol require the reagents listed in <A href="../01/23.htm">Chapter 
      1, Protocol 23</A>, <A href="../01/24.htm">Chapter 
      1, Protocol 24</A>, or <A href="../01/26.htm">Chapter 
      1, Protocol 26</A> for the transformation of <I>E. coli</I>.</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Steps 10 and 
      16 of this protocol require the reagents listed in <A href="../01/01.htm">Chapter 
      1, Protocol 1</A> for the minipreparation of plasmid DNA.</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 17 of 
      this protocol requires the reagents listed in <A href="../12/03.htm">Chapter 
      12, Protocol 3</A>, <A href="../12/04.htm">Chapter 
      12, Protocol 4</A>, or <A href="../12/05.htm">Chapter 
      12, Protocol 5</A>.</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Vectors and Bacterial Strains</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>E. coli</I> 
      strain with a <I>mutS</I> genotype (e.g., BMH 71-18) competent for 
      transformation</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>E. coli</I> 
      strain with a <I>mut<SUP>+</SUP></I> phenotype competent for 
      transformation</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Please see 
      Step 14.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 
  border=0></TD></TR></TBODY></TABLE><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" 
size=3><STRONG>METHOD</STRONG></FONT> <BR><FONT face="arial,verdana, helvetica" 
color=#000000 size=2>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Mix the 
  following components in a microfuge tube: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10x annealing 
        buffer (13-4)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>plasmid 
        DNA</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>0.025 to 0.25 
        pmole</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>selection 
        primer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>25 
      pmoles</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>mutagenic 
        primer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>25 
      pmoles</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>to 20 
      µl</FONT></TD></TR></TBODY></TABLE>Incubate the tube in a boiling water bath 
  for 5 minutes. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Immediately 
  chill the tube in ice for 5 minutes. Centrifuge the tube for 5 seconds in a 
  microfuge to deposit the fluid at the base. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>To the tube of 
  annealed primers and plasmid, add: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10x synthesis 
        buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>3 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>bacteriophage 
        T4 DNA polymerase (2-4 units/µl)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>bacteriophage 
        T4 DNA ligase (4-6 units/µl)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5 
    µl</FONT></TD></TR></TBODY></TABLE>Mix the reagents well by gentle up and down 
  pipetting. Centrifuge the tube for 5 seconds in a microfuge to deposit the 
  fluid at the base. Incubate the reaction for 1-2 hours at 37°C. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Stop the 
  reaction by heating the tube for at least 5 minutes at 70°C to inactivate the 
  enzymes. Store the tube on the bench to allow it cool to room temperature. 
  </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Adjust the NaCl 
  concentration of the reaction to a level that is optimal for the selected 
  unique site restriction endonuclease. Use the 10x annealing buffer, a stock of 
  NaCl, or the 10x buffer supplied with the restriction enzyme. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Add 20 units of 
  the selective restriction endonuclease to the reaction mixture. Incubate the 
  reaction for at least 1 hour at the appropriate digestion 
  temperature.<BR><B>IMPORTANT</B><I> The volume of enzymes added to the 
  reactions (including polymerase and ligase) should not exceed 10% of the total 
  reaction volume. Adjust the reaction volume accordingly.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Transform a 
  <I>mutS E. coli</I> strain such as BMH 71-18 with the plasmid DNAs contained 
  in the digestion mixture, using one of the transformation procedures described 
  in <A href="../01/23.htm">Chapter 
  1, Protocol 23</A> to <A href="../01/26.htm">Chapter 
  1, Protocol 26</A>. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Spread 10, 50, 
  and 250 µl of the transformation mixture onto LB agar plates containing the 
  appropriate antibiotic. Incubate the plates overnight at 37°C. Carry out Step 
  9 while the plates are incubating. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Amplify the 
  plasmids by adding the remaining transformation mixture to 3 ml of LB medium 
  containing the appropriate antibiotic. Incubate the culture overnight at 37°C 
  with shaking. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>The next day, 
  prepare plasmid DNA from approx. 2.5 ml of the overnight culture (please see 
  <A href="../01/01.htm">Chapter 
  1, Protocol 1</A>). </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Digest the 
  plasmid DNA prepared in Step 10 with the selective restriction enzyme: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>plasmid 
        DNA</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>500 
    ng</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10x restriction 
        enzyme buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>unique site 
        restriction endonuclease</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>20 
      units</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>to 20 
      µl</FONT></TD></TR></TBODY></TABLE>Incubate the reaction for 2 hours at the 
  appropriate temperature. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Add an 
  additional 10 units of the restriction enzyme, and incubate for at least 1 
  further hour. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Assess the 
  extent of digestion by running 5-10 µl of the plasmid DNA on a 1% agarose gel 
  containing 0.5 µg/ml ethidium bromide. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Transform 
  competent <I>mutS</I><SUP>+</SUP> <I>E. coli</I> cells with either 2-4 µl of 
  the digested plasmid DNA (approx. 50-100 ng) for transformation of chemically 
  treated competent cells or 1 µl of plasmid DNA diluted fivefold with sterile 
  H<SUB>2</SUB>O (approx. 5 ng) for transformation by electroporation (please 
  see <A href="../01/23.htm">Chapter 
  1, Protocol 23</A> to <A href="../01/26.htm">Chapter 
  1, Protocol 26</A>). </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Spread 10, 50, 
  and 250 µl of the transformation mixture onto LB agar plates containing the 
  appropriate antibiotic. Incubate the plates overnight at 37°C. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>The next day, 
  prepare minipreparations of plasmid DNA from at least 12 independent 
  transformants. Screen the preparations by restriction endonuclease digestion 
  and agarose gel electrophoresis to identify plasmids that are resistant to 
  cleavage by the selective restriction enzyme. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Use DNA 
  sequencing to confirm that the plasmids contain the desired mutation (please 
  see <A href="../12/03.htm">Chapter 
  12, Protocol 3</A>, <A href="../12/04.htm">Chapter 
  12, Protocol 4</A>, or <A href="../12/05.htm">Chapter 
  12, Protocol 5</A>). </FONT></LI></OL></FONT><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>RECIPES</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/circle.gif" width=12 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>ATP</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve an appropriate amount of solid ATP in 25 mM Tris-Cl (pH 
      8.0).</P>
      <P>Store the 10 mM ATP solution in small aliquots at 
  -20°C.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Annealing buffer (13-4)</STRONG></FONT> 
</TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>200 mM Tris-Cl (pH 
      7.5)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>100 mM 
      MgCl<SUB>2</SUB></FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>500 mM 
  NaCl</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>LB</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>deionized H<SUB>2</SUB>O, 
      to 950 ml</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>tryptone, 10 
  g</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>yeast extract, 5 
    g</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>NaCl, 10 g</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>For solid medium, please 
      see Media Containing Agar or Agarose.</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare LB 
      (Luria-Bertani medium), shake until the solutes have dissolved. Adjust the 
      pH to 7.0 with 5 N NaOH (approx. 0.2 ml). Adjust the volume of the 
      solution to 1 liter with deionized H<SUB>2</SUB>O. Sterilize by 
      autoclaving for 20 minutes at 15 psi (1.05 kg/cm <SUP>2</SUP>) on liquid 
      cycle.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Media Containing Agar or 
      Agarose</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Prepare liquid media according to the recipes given. Just before 
      autoclaving, add one of the following:</P>
      <P>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face="arial, helvetica,verdana" size=2>Bacto Agar (for 
            plates)</FONT></TD>
          <TD><FONT face="arial, helvetica,verdana" size=2>15 
        g/liter</FONT></TD></TR>
        <TR>
          <TD><FONT face="arial, helvetica,verdana" size=2>Bacto Agar (for top 
            agar)</FONT></TD>
          <TD><FONT face="arial, helvetica,verdana" size=2>7 
        g/liter</FONT></TD></TR>
        <TR>
          <TD><FONT face="arial, helvetica,verdana" size=2>agarose (for 
            plates)</FONT></TD>
          <TD><FONT face="arial, helvetica,verdana" size=2>15 
        g/liter</FONT></TD></TR>
        <TR>
          <TD><FONT face="arial, helvetica,verdana" size=2>agarose (for top 
            agarose)</FONT></TD>
          <TD><FONT face="arial, helvetica,verdana" size=2>7 
        g/liter</FONT></TD></TR></TBODY></TABLE></P>
      <P>Sterilize by autoclaving for 20 minutes at 15 psi (1.05 kg/cm 
      <SUP>2</SUP>) on liquid cycle. When the medium is removed from the 
      autoclave, swirl it gently to distribute the melted agar or agarose evenly 
      throughout the solution. <I>Be careful!</I> The fluid may be superheated 
      and may boil over when swirled. Allow the medium to cool to 50-60°C before 
      adding thermolabile substances (e.g., antibiotics). To avoid producing air 
      bubbles, mix the medium by swirling. Plates can then be poured directly 
      from the flask; allow approx. 30-35 ml of medium per 90-mm plate. To 
      remove bubbles from medium in the plate, flame the surface of the medium 
      with a Bunsen burner before the agar or agarose hardens. Set up a color 
      code (e.g., two red stripes for LB-ampicillin plates; one black stripe for 
      LB plates, etc.) and mark the edges of the plates with the appropriate 
      colored markers.</P>
      <P>When the medium has hardened completely, invert the plates and store 
      them at 4°C until needed. The plates should be removed from storage 1-2 
      hours before they are used. If the plates are fresh, they will "sweat" 
      when incubated at 37°C. When this condensation drops on the agar/agarose 
      surface, it allows bacterial colonies or bacteriophage plaques to spread 
      and increases the chances of cross-contamination. This problem can be 
      avoided by wiping off the condensation from the lids of the plates and 
      then incubating the plates for several hours at 37°C in an inverted 
      position before they are used. Alternatively, remove the liquid by shaking 
      the lid with a single, quick motion. To minimize the possibility of 
      contamination, hold the open plate in an inverted position while removing 
      the liquid from the lid.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>NaCl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare a 5 M 
      solution: Dissolve 292 g of NaCl in 800 ml of H<SUB>2</SUB>O. Adjust the 
      volume to 1 liter with H<SUB>2</SUB>O. Dispense into aliquots and 
      sterilize by autoclaving. Store the NaCl solution at room 
      temperature.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Synthesis buffer</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>100 mM Tris-Cl (pH 
      7.5)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>dTTP, dATP, dCTP, and 
      dGTP, each at a concentration of 5 mM</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>10 mM ATP</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>20 mM 
      dithiothreitol</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>For a 10x 
    buffer.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Tris-Cl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve 121.1 g of Tris base in 800 ml of H<SUB>2</SUB>O. Adjust the 
      pH to the desired value by adding concentrated HCl.</P>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2><I>pH</I></FONT> </TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2><I>HCl</I></FONT> 
        </TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.4 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>70 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.6 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>60 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>8.0 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>42 ml 
        </FONT></TD></TR></TBODY></TABLE>
      <P>(1 M) Allow the solution to cool to room temperature before making 
      final adjustments to the pH. Adjust the volume of the solution to 1 liter 
      with H<SUB>2</SUB>O. Dispense into aliquots and sterilize by 
      autoclaving.</P>
      <P>If the 1 M solution has a yellow color, discard it and obtain Tris of 
      better quality. The pH of Tris solutions is temperature-dependent and 
      decreases approx. 0.03 pH units for each 1°C increase in temperature. For 
      example, a 0.05 M solution has pH values of 9.5, 8.9, and 8.6 at 5°C, 
      25°C, and 37°C, 
respectively.</P></FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;     helvetica" size=3><STRONG>CAUTIONS</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" width=11 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Radioactive substances</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Radioactive substances: When planning an experiment that involves 
      the use of radioactivity, consider the physico-chemical properties of the 
      isotope (half-life, emission type, and energy), the chemical form of the 
      radioactivity, its radioactive concentration (specific activity), total 
      amount, and its chemical concentration. Order and use only as much as 
      needed. Always wear appropriate gloves, lab coat, and safety goggles when 
      handling radioactive material. X-rays and gamma rays are electromagnetic 
      waves of very short wavelengths either generated by technical devices or 
      emitted by radioactive materials. They might be emitted isotropically from 
      the source or may be focused into a beam. Their potential dangers depend 
      on the time period of exposure, the intensity experienced, and the 
      wavelengths used. Be aware that appropriate shielding is usually made of 
      lead or other similar material. The thickness of the shielding is 
      determined by the energy(s) of the X-rays or gamma rays. Consult the local 
      safety office for further guidance in the appropriate use and disposal of 
      radioactive materials. Always monitor thoroughly after using 
      radioisotopes. A convenient calculator to perform routine radioactivity 
      calculations can be found at: <A target=_blank 
      href="http://www.graphpad.com/calculators/radcalc.cfm">http://www.graphpad.com/calculators/radcalc.cfm</A>. 
      </FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>REFERENCES</STRONG> 
</FONT><BR>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Deng W.P. and 
  Nickoloff J.A. 1992. Site-directed mutagenesis of virtually any plasmid by 
  eliminating a unique site. <I>Anal. Biochem.</I> 200:81-88. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Zhu L. 1996. In 
  vitro site-directed mutagenesis using the unique restriction site elimination 
  (USE) method. <I>Methods Mol. Biol.</I> 57:13-29. 
</FONT></LI></OL><BR><BR><BR><BR><BR><FONT face=Arial,Verdana,Helvetica 
size=1><STRONG>Copyright © 2000 by Cold Spring Harbor Laboratory Press. All 
rights reserved.</STRONG><BR>No part of these pages, either text or image may be 
used for any purpose other than personal use. Therefore, reproduction 
modification, storage in a retrieval system or retransmission, in any form or by 
any means, electronic, mechanical, or otherwise, for reasons other than personal 
use, is strictly prohibited without prior written 
permission.</FONT><BR><BR>
</td></tr></table>
</BODY></HTML>
