<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Molecular Cloning | Protocol 6 Print Version</TITLE>
<META http-equiv=Content-Type content="text/html; charset=windows-1251"><!--================== Designed and Developed by WriteDesign =================--><!--======================= Powered by RomeBlack, Inc. =======================--><!--=== Copyright 2000, 2001 Cold Spring Harbor Labs. All Rights Reserved. ===-->
<META content="" name=keywords>
<META 
content="Four primers and three PCRs are used to create a site-specific mutation by overlap extension. One pair of primers is used to amplify DNA that contains the mutation site together with upstream sequences. The second pair of primers is used in a separate PCR to amplify DNA that contains the mutation site together with downstream sequences. The mutation(s) of interest is located in the region of overlap and therefore in both amplified fragments. The overlapping fragments are mixed, denatured, and annealed to generate heteroduplexes that can be extended and, in a third PCR, amplified into a full-length DNA using two primers that bind to the extremes of the two initial fragments." 
name=description>
<META content="MSHTML 5.50.4134.600" name=GENERATOR></HEAD>
<BODY bgColor=#ffffff background=../izo/fon.gif>
<table width=600 border="0" cellspacing="0" cellpadding="0"><tr><td width=600><IMG height=49 hspace=4 src="../izo/logo_csh.gif" 
width=169 align=left vspace=4><IMG height=49 hspace=4 
src="../izo/print_header.gif" width=293 align=left vspace=4><BR 
clear=all><BR><FONT face="arial,verdana, helvetica" size=4><STRONG>Chapter 13, 
Protocol 6</STRONG></FONT><BR><BR clear=all><FONT face=arial,verdana,helvetica 
size=3><STRONG>Site-specific Mutagenesis by Overlap 
Extension</STRONG></FONT><BR><BR><FONT face="arial,verdana, helvetica" 
size=2>Four primers and three PCRs are used to create a site-specific mutation 
by overlap extension. One pair of primers is used to amplify DNA that contains 
the mutation site together with upstream sequences. The second pair of primers 
is used in a separate PCR to amplify DNA that contains the mutation site 
together with downstream sequences. The mutation(s) of interest is located in 
the region of overlap and therefore in both amplified fragments. The overlapping 
fragments are mixed, denatured, and annealed to generate heteroduplexes that can 
be extended and, in a third PCR, amplified into a full-length DNA using two 
primers that bind to the extremes of the two initial fragments.</FONT> <BR><BR>
<TABLE border=1>
  <TBODY>
  <TR>
    <TD width=700><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" 
      width=11 border=0><FONT face=arial,verdana,helvetica 
      size=2><STRONG>CAUTION</STRONG></FONT><BR><IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0><FONT 
      face="arial,verdana, helvetica" size=2> <STRONG>RECIPE</STRONG></FONT> 
  </TD></TR></TBODY></TABLE><BR><BR><FONT face=arial,verdana,helvetica 
size=3><STRONG>MATERIALS</STRONG></FONT><BR><BR>
<TABLE cellPadding=0 cellborder="0">
  <TBODY>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Buffers and Solutions</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>10x 
      Amplification buffer</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>dNTP solution 
      containing all four dNTPs, each at 20 mM</FONT> <IMG height=12 alt="" 
      hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Enzymes and Buffers</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Thermostable 
      DNA polymerase</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>To avoid 
      the introduction of erroneous bases, use a highly processive thermostable 
      DNA polymerase with 3&acute;-5&acute; exonuclease "proofreading" capacity in overlap 
      extension mutagenesis. In addition, the DNA polymerase used must not 
      catalyze the nontemplate addition of adenine residues. Thermostable DNA 
      polymerases with the appropriate properties include </I>Pwo<I> DNA 
      polymerase (Boehringer Mannheim), r</I>Tth<I> DNA polymerase XL 
      (Perkin-Elmer), </I>VentR<I> DNA polymerase (New England Biolabs), and 
      </I>Pfu<I> DNA polymerase (Stratagene).</I><BR><I>Mixes of thermostable 
      DNA polymerases designed for use in long PCR are also well suited for 
      overlap extension mutagenesis.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Nucleic Acids and Oligonucleotides</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>Oligonucleotide Primers</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Each primer 
      should be 20-30 nucleotides in length and contain approximately equal 
      numbers of the four bases, with a balanced distribution of G and C 
      residues and a low propensity to form stable secondary structure. The 
      sequences of the mutagenic oligonucleotide primers FM and RM should share 
      at least a 15-bp overlap, and the mismatched base pairs within these 
      primers should be located in the center of the oligonucleotide. The primer 
      sequences at the 5&acute; and 3&acute; ends of the DNA fragment to be amplified (i.e., 
      those with wild-type sequences R2, F2) can incorporate unique restriction 
      endonuclease cleavage sites to aid in the subsequent cloning of the 
      mutagenized DNA fragment. For general features of primer design, please 
      see the discussion on Design of Oligonucleotide Primers in the 
      introduction to Chapter 10 in the print version of the 
      manual.</I><BR><I>Oligonucleotide primers synthesized on an automated DNA 
      synthesizer can generally be used in overlap extension mutagenesis without 
      further purification.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Template 
      DNA</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>The 
      template DNA used for mutagenesis is usually a plasmid DNA containing the 
      gene or cDNA of interest. Dissolve the DNA at 1 µg/ml in 10 mM Tris-Cl (pH 
      7.6) containing a low concentration of EDTA (&lt;0.1 mM).</I></FONT> 
</TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Additional Reagents</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 11 of 
      this protocol requires the reagents listed in <A href="../01/17.htm">Chapter 
      1, Protocol 17</A> or <A href="../01/19.htm">Chapter 
      1 Protocol 19.</A></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 12 of 
      this protocol requires the reagents listed in <A href="../12/03.htm">Chapter 
      12, Protocol 3</A>, <A href="../12/04.htm">Chapter 
      12, Protocol 4</A>, or <A href="../12/05.htm">Chapter 
      12, Protocol 5</A>.</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 
  border=0></TD></TR></TBODY></TABLE><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" 
size=3><STRONG>METHOD</STRONG></FONT> <BR><FONT face="arial,verdana, helvetica" 
color=#000000 size=2>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Design and 
  synthesize oligonucleotide primers FM, RM, R2, and F2 based on the known 
  sequence of the DNA, as outlined in the protocol introduction and Materials 
  section. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>In a sterile 
  0.5-ml microfuge tube or amplification tube, set up PCR 1 by mixing the 
  following reagents: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>template 
        DNA</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>approx. 100 
        ng</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10x 
        amplification buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>20 mM mixture 
        of four dNTPs</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1.0 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5 µM primer RM 
        (30 pmoles)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>6.0 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5 µM primer F2 
        (30 pmoles)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>6.0 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>thermostable 
        DNA polymerase</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1-2 
      units</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>to 100 
      µl</FONT></TD></TR></TBODY></TABLE></FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>In a second 
  sterile 0.5-ml microfuge tube or amplification tube, set up PCR 2 by mixing 
  the following reagents: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>template 
        DNA</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>approx. 100 
        ng</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10x 
        amplification buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>20 mM mixture 
        of four dNTPs</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1.0 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5 µM primer FM 
        (30 pmoles)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>6.0 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5 µM primer R2 
        (30 pmoles)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>6.0 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>thermostable 
        DNA polymerase</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1-2 
      units</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>to 100 
      µl</FONT></TD></TR></TBODY></TABLE></FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>If the 
  thermocycler does not have a heated lid, overlay the PCRs with 1 drop (approx. 
  50 µl) of light mineral oil. Place the tubes in a thermocycler. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Amplify the 
  nucleic acids using the denaturation, annealing, and polymerization times and 
  temperatures listed in the table below. 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2><B>Cycle 
        Number</B></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2><B>Denaturation</B></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2><B>Annealing</B></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2><B>Polymerization</B></FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>20 
      cycles</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 min at 
        94°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 min at 
        50°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1-3 min at 
        72°C</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Last 
        cycle</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 min at 
        94°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10 min at 
        72°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
    size=2></FONT></TD></TR></TBODY></TABLE><I>Times and temperatures may need to 
  be adapted to suit the particular reaction conditions.</I><BR><I>The length of 
  the polymerization step should be calculated from the polymerization rate of 
  the thermostable DNA polymerase employed and the length of the DNA template to 
  be amplified.</I><BR><I>The temperature of the annealing reaction may have to 
  be adjusted depending on the sequence of the mutagenic primers.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Analyze 5% of 
  each of the two PCRs on an agarose or polyacrylamide gel and estimate the 
  concentration of amplified target DNAs. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 
  size=2>(<I>Optional</I>) Purify the two PCR products using one of the 
  protocols described in Chapter 5. Including this purification step often 
  increases the yield of the desired amplification product in Step 8 of the 
  protocol and reduces the background of spurious amplification products. 
</FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>In a sterile 
  0.5-ml microfuge tube or amplification tube, mix the following reagents in an 
  amplification reaction to join the 5&acute; and 3&acute; ends of the target gene: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>amplification 
        product PCR 1 (Step 2)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>approx. 50 
        ng</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>amplification 
        product PCR 2 (Step 3)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>approx. 50 
        ng</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10x 
        amplification buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5 µM primer F2 
        (30 pmoles)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>6.0 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5 µM primer R2 
        (30 pmoles)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>6.0 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>thermostable 
        DNA polymerase</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1-2 
      units</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>to 100 
      µl</FONT></TD></TR></TBODY></TABLE></FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>If the 
  thermocycler does not have a heated lid, overlay the PCRs with 1 drop (approx. 
  50 µl) of light mineral oil. Place the tubes in a thermocycler. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Amplify the 
  nucleic acids using the denaturation, annealing, and polymerization times and 
  temperatures listed above in Step 5. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Analyze 5% of 
  the PCR on an agarose or polyacrylamide acrylamide gel and estimate the 
  concentration of amplified target DNA. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Verify the 
  complete sequence of the amplified DNA fragment after cloning to ensure that 
  no mutations other than those in primers FM and RM were introduced during 
  these manipulations. </FONT></LI></OL></FONT><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>RECIPES</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/circle.gif" width=12 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Amplification Buffer</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>500 mM KCl</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>100 mM Tris-Cl (pH 8.3 at 
      room temperature)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>15 mM 
      MgCl<SUB>2</SUB></FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Autoclave the 10x buffer 
      for 10 minutes at 15 psi (1.05 kg/cm <SUP>2</SUP>) on liquid cycle. Divide 
      the sterile buffer into aliquots and store them at 
  -20°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>KCl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Dissolve an appropriate 
      amount of solid KCl in H<SUB>2</SUB>O, autoclave for 20 minutes on liquid 
      cycle and store at room temperature. Ideally, this 4 M solution should be 
      divided into small (approx. 100 µl) aliquots in sterile tubes and each 
      aliquot thereafter used one time.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Tris-Cl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve 121.1 g of Tris base in 800 ml of H<SUB>2</SUB>O. Adjust the 
      pH to the desired value by adding concentrated HCl.</P>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2><I>pH</I></FONT> </TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2><I>HCl</I></FONT> 
        </TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.4 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>70 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.6 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>60 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>8.0 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>42 ml 
        </FONT></TD></TR></TBODY></TABLE>
      <P>(1 M) Allow the solution to cool to room temperature before making 
      final adjustments to the pH. Adjust the volume of the solution to 1 liter 
      with H<SUB>2</SUB>O. Dispense into aliquots and sterilize by 
      autoclaving.</P>
      <P>If the 1 M solution has a yellow color, discard it and obtain Tris of 
      better quality. The pH of Tris solutions is temperature-dependent and 
      decreases approx. 0.03 pH units for each 1°C increase in temperature. For 
      example, a 0.05 M solution has pH values of 9.5, 8.9, and 8.6 at 5°C, 
      25°C, and 37°C, respectively.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>dNTP Solution</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve each dNTP (deoxyribonucleoside triphosphates) in 
      H<SUB>2</SUB>O at an approximate concentration of 100 mM. Use 0.05 M Tris 
      base and a micropipette to adjust the pH of each of the solutions to 7.0 
      (use pH paper to check the pH). Dilute an aliquot of the neutralized dNTP 
      appropriately, and read the optical density at the wavelengths given in 
      the table below. Calculate the actual concentration of each dNTP. Dilute 
      the solutions with H<SUB>2</SUB>O to a final concentration of 50 mM dNTP. 
      Store each separately at -70°C in small aliquots. For polymerase chain 
      reactions (PCRs), adjust the dNTP solution to pH 8.0 with 2 N NaOH. 
      Commercially available solutions of PCR-grade dNTPs require no 
      adjustment.</P>
      <P>
      <TABLE width=400>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>Base</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana 
            size=2>wavelength<BR>(nm)</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>Extinction 
            Coefficient<BR>(E)&nbsp;(M<SUP>-1</SUP>cm<SUP>-1</SUP>)</FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>A</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>259</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.54 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>G</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>253</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.37 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>C</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>271</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.10 x 
            10<SUP>3</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>T</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>267</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.60 x 
            10<SUP>3</SUP></FONT></TD></TR></TBODY></TABLE></P>
      <P>For a cuvette with a path length of 1 cm, absorbance = EM. 100 mM stock 
      solutions of each dNTP are commercially available 
    (Pharmacia).</P></FONT></TD></TR></TBODY></TABLE><BR><BR><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>REFERENCES</STRONG> 
</FONT><BR>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Higuchi R., 
  Krummel B., and Saiki R.K. 1988. A general method of <I>in vitro</I> 
  preparation and specific mutagenesis of DNA fragments: Study of protein and 
  DNA interactions. <I>Nucleic Acids Res.</I> 16:7351-7367. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Ho S.N., Hunt 
  H.D., Horton R.M., Pullen J.K., and Pease L.R. 1989. Site-directed mutagenesis 
  by overlap extension using the polymerase chain reaction. <I>Gene</I> 
  77:51-59. </FONT></LI></OL><BR><BR><BR><BR><BR><FONT 
face=Arial,Verdana,Helvetica size=1><STRONG>Copyright © 2000 by Cold Spring 
Harbor Laboratory Press. All rights reserved.</STRONG><BR>No part of these 
pages, either text or image may be used for any purpose other than personal use. 
Therefore, reproduction modification, storage in a retrieval system or 
retransmission, in any form or by any means, electronic, mechanical, or 
otherwise, for reasons other than personal use, is strictly prohibited without 
prior written permission.</FONT><BR><BR>
</td></tr></table>
</BODY></HTML>
