<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Molecular Cloning | Protocol 8 Print Version</TITLE>
<META http-equiv=Content-Type content="text/html; charset=windows-1251"><!--================== Designed and Developed by WriteDesign =================--><!--======================= Powered by RomeBlack, Inc. =======================--><!--=== Copyright 2000, 2001 Cold Spring Harbor Labs. All Rights Reserved. ===-->
<META content="" name=keywords>
<META 
content="In this method, an oligodeoxynucleotide primer hybridized to mRNA is extended by an RNA-dependent DNA polymerase to create a cDNA copy that can be amplified by PCR. Depending on the purpose of the experiment, the primer for first-strand cDNA synthesis can be specifically designed to hybridize to a particular target gene, or a general primer such as oligo(dT) can be used to prime cDNA synthesis from essentially all mammalian mRNAs. Similarly, the reverse primer used in the subsequent amplification reaction can be gene-specific or general (e.g., random hexamers). To reduce the chance of contamination with exogenous DNAs, prepare and use a special set of reagents and solutions for PCR only. Bake all glassware for 6 hours at 150°C and autoclave all plasticware." 
name=description>
<META content="MSHTML 5.50.4134.600" name=GENERATOR></HEAD>
<BODY bgColor=#ffffff background=../izo/fon.gif>
<table width=600 border="0" cellspacing="0" cellpadding="0"><tr><td width=600><IMG height=49 hspace=4 src="../izo/logo_csh.gif" 
width=169 align=left vspace=4><IMG height=49 hspace=4 
src="../izo/print_header.gif" width=293 align=left vspace=4><BR 
clear=all><BR><FONT face="arial,verdana, helvetica" size=4><STRONG>Chapter 8, 
Protocol 8</STRONG></FONT><BR><BR clear=all><FONT face=arial,verdana,helvetica 
size=3><STRONG>Amplification of cDNA Generated by Reverse Transcription of 
mRNA</STRONG></FONT><BR><BR><FONT face="arial,verdana, helvetica" size=2>In this 
method, an oligodeoxynucleotide primer hybridized to mRNA is extended by an 
RNA-dependent DNA polymerase to create a cDNA copy that can be amplified by PCR. 
Depending on the purpose of the experiment, the primer for first-strand cDNA 
synthesis can be specifically designed to hybridize to a particular target gene, 
or a general primer such as oligo(dT) can be used to prime cDNA synthesis from 
essentially all mammalian mRNAs. Similarly, the reverse primer used in the 
subsequent amplification reaction can be gene-specific or general (e.g., random 
hexamers). To reduce the chance of contamination with exogenous DNAs, prepare 
and use a special set of reagents and solutions for PCR only. Bake all glassware 
for 6 hours at 150°C and autoclave all plasticware.</FONT> <BR><BR>
<TABLE border=1>
  <TBODY>
  <TR>
    <TD width=700><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" 
      width=11 border=0><FONT face=arial,verdana,helvetica 
      size=2><STRONG>CAUTION</STRONG></FONT><BR><IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0><FONT 
      face="arial,verdana, helvetica" size=2> <STRONG>RECIPE</STRONG></FONT> 
  </TD></TR></TBODY></TABLE><BR><BR><FONT face=arial,verdana,helvetica 
size=3><STRONG>MATERIALS</STRONG></FONT><BR><BR>
<TABLE cellPadding=0 cellborder="0">
  <TBODY>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Buffers and Solutions</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>10x 
      Amplification buffer</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>Chloroform</FONT> <IMG height=11 alt=caution hspace=4 
      src="../izo/triangle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>dNTP solution 
      (20 mM) containing all four dNTPs (pH 8.0)</FONT> <IMG height=12 alt="" 
      hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Ethanol</FONT> 
    </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>MgCl<SUB>2</SUB> (50 mM)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>Phenol:chloroform (1:1, v/v) </FONT><IMG height=11 alt=caution 
      hspace=4 src="../izo/triangle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Placental 
      RNase inhibitor (20 units/µl)</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Enzymes and Buffers</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Reverse 
      transcriptase (RNA-dependent DNA polymerase)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Thermostable 
      DNA-dependent DNA polymerase</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Taq <I>DNA 
      polymerase is the standard and appropriate enzyme for the amplification 
      stage of most forms of RT-PCR. However, where elongation from 
      3&acute;-mismatched primers is suspected, a thermostable DNA polymerase with 
      3&acute;-&gt;5&acute; proofreading activity may be preferred.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Nucleic Acids and Oligonucleotides</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Exogenous 
      reference RNA</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Please see 
      Step 2.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Gene-specific 
      oligonucleotide (20 µM) in H<SUB>2</SUB>O (20 pmoles/µl)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>The 
      gene-specific oligonucleotide should be complementary to a known sequence 
      in the target mRNA.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>Oligo(dT)<SUB>12-18</SUB> (100 µg/ml) in TE (pH 8.0)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Random 
      hexanucleotides (1 mg/ml) in TE (pH 8.0)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Sense and 
      antisense oligonucleotide primers for amplification of cDNA by PCR</FONT> 
    </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>The primer 
      used to generate cDNA may also be used as the antisense primer in the 
      amplification stage of standard RT-PCR. However, the specificity of 
      amplification can be improved by using an antisense primer that binds to 
      an upstream sequence in the target transcript. Both sense and antisense 
      primers are gene-specific synthetic oligonucleotides, which should be 
      20-30 nucleotides in length and contain approximately equal numbers of the 
      four bases, with a balanced distribution of G and C residues, and a low 
      propensity to form stable secondary structures.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Template 
      RNA</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>The 
      template may be total RNA (100 µg/ml) in H<SUB>2</SUB>O or 
      poly(A)<SUP>+</SUP> RNA (10 µg/ml) in H<SUB>2</SUB>O. Total RNA extracted 
      from cells with chaotropic agents is generally the template of choice for 
      RT-PCR for mRNAs that are expressed at moderate to high abundance. 
      Poly(A)<SUP>+</SUP> is preferred as a template for all forms of RT-PCR 
      when the target mRNA is expressed at low abundance.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Additional Reagents</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 8 of this 
      protocol requires the reagents listed in <A href="../05/02.htm">Chapter 
      5, Protocol 2 </A>, <A href="../12/06.htm">Chapter 
      12, Protocol 6</A>, or <A href="../06/10.htm">Chapter 
      6, Protocol 10 </A>.</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 10 of 
      this protocol requires the reagents listed in <A href="03.htm">Chapter 
      8, Protocol 3 </A>and <A href="04.htm">Chapter 
      8, Protocol 4, </A><A href="05.htm">Chapter 
      8, Protocol 5 </A>, or <A href="06.htm">Chapter 
      8, Protocol 6 </A>.</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 
  border=0></TD></TR></TBODY></TABLE><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" 
size=3><STRONG>METHOD</STRONG></FONT> <BR><FONT face="arial,verdana, helvetica" 
color=#000000 size=2>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Transfer 1 pg 
  to 100 ng of poly(A)<SUP>+</SUP> mRNA or 10 pg to 1 µg of total RNA to a fresh 
  microfuge tube. Adjust the volume to 10 µl with H<SUB>2</SUB>O. Denature by 
  heating at 75°C for 5 minutes, followed by chilling on ice. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>To the 
  denatured RNA add: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10x 
        amplification buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>20 mM solution 
        of four dNTPs (pH 8.0)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>primers 
        (optimized, please see below)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>approx. 20 
        units/µl placental RNase inhibitor</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>50 mM 
        MgCl<SUB>2</SUB></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>100-200 
        units/µl reverse transcriptase</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>to 20 
      µl</FONT></TD></TR></TBODY></TABLE>Incubate the reaction for 60 minutes at 
  37°C.<BR><I>Depending on the experiment, oligo(dT)<SUB>12-18</SUB>, random 
  hexanucleotides, or gene-specific antisense oligonucleotides can be used as 
  primers for synthesis of first-strand cDNA.</I><BR><I>The optimum ratio of 
  primer to template should be ascertained empirically for each preparation of 
  RNA. As a starting point for optimization, we recommend adding varying amounts 
  of primers to 20-µl reactions:</I> 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>synthetic 
        oligonucleotide complementary to the target RNA:</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5-20 
        pmoles</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>oligo(dT)<SUB>12-18</SUB>:</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>0.1-0.5 
        µg</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>random 
        hexanucleotides:</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1-5 
    µg</FONT></TD></TR></TBODY></TABLE>Set up three negative control reactions. In 
  one reaction, include all components of the first-strand reaction except the 
  RNA template. In another reaction, include all components except the reverse 
  transcriptase. Omit primers from the third reaction. Carry the controls 
  through all subsequent steps of the protocol. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Inactivate the 
  reverse transcriptase and denature the template-cDNA complexes by heating the 
  reaction to 95°C for 5 minutes or by phenol extraction and ethanol 
  precipitation. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Adjust the 
  reaction mixture so that it contains 20 pmoles of the sense and antisense 
  primers. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Add to the 
  reaction mixture: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1x 
        amplification buffer (or volume<BR>required to bring reaction mixture to 
        99 µl)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>77 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1-2 units 
        thermostable DNA polymerase</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR></TBODY></TABLE></FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>If the thermal 
  cycler is not fitted with a heated lid, overlay the reaction mixtures with 1 
  drop (approx. 50 µl) of light mineral oil. Alternatively, place a bead of 
  paraffin wax into the tube if using a hot start protocol. Place the tubes or 
  the microtiter plate in the thermal cycler. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Amplify the 
  nucleic acids using the denaturation, annealing, and polymerization times and 
  temperatures listed in the table. 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2><B>Cycle 
        Number</B></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2><B>Denaturation</B></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2><B>Annealing</B></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2><B>Polymerization</B></FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>35 
      Cycles</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>45 sec at 
        94°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>45 sec at 
        55°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 min 15 sec at 
        72°C</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Last 
        cycle</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 min at 
        94°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>45 sec at 
        55°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 min 15 sec at 
        72°C</FONT></TD></TR></TBODY></TABLE><I>Times and temperatures may need to be 
  adapted to suit the particular reaction conditions.</I><BR><I>Polymerization 
  should be carried out for 1 minute for every 1000 bp of length of the target 
  DNA.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Withdraw a 
  sample (5-10 µl) from the test reaction mixture and the four control reactions 
  and analyze them by electrophoresis through an agarose or polyacrylamide gel. 
  Include DNA markers of an appropriate size. Stain the gel with ethidium 
  bromide or SYBR Gold to visualize the DNA.<BR><I>A successful amplification 
  reaction should yield a readily visible DNA fragment of the expected size. The 
  identity of the band can be confirmed by DNA sequencing (please see <A href="../12/06.htm">Chapter 
  12, Protocol 6</A>), Southern hybridization (please see <A href="../06/10.htm">Chapter 
  6, Protocol 10 </A>), and/or restriction mapping.</I><BR><I>If no product is 
  visible after 30 cycles of amplification, add fresh </I>Taq<I> polymerase and 
  continue the amplification reaction for a further 15-20 cycles.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>If mineral oil 
  was used to overlay the reaction (in Step 6), remove the oil from the sample 
  before cloning by extraction with 150 µl of chloroform.<BR><I>The aqueous 
  phase, which contains the amplified DNA, will form a micelle near the 
  meniscus. The micelle can then be transferred to a fresh tube with an 
  automatic micropipette.</I><BR><B>IMPORTANT</B> <I>Do not attempt chloroform 
  extractions in microtiter plates. The plastic used in these plates is not 
  resistant to organic solvents.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Clone the 
  amplified products into an appropriately prepared vector by any of the methods 
  described in <A href="04.htm">Chapter 
  8, Protocol 4 </A>, <A href="05.htm">Chapter 
  8, Protocol 5 </A>, or <A href="06.htm">Chapter 
  8, Protocol 6 </A>. Before cloning, separate the amplified DNA from the 
  residual thermostable DNA polymerase and dNTPs (please see <A href="03.htm">Chapter 
  8, Protocol 3</A>). The DNA can then be ligated to a blunt-ended vector or a T 
  vector, or it can digested with restriction enzymes and ligated to a vector 
  with compatible termini. </FONT></LI></OL></FONT><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>RECIPES</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/circle.gif" width=12 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Amplification Buffer</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>500 mM KCl</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>100 mM Tris-Cl (pH 8.3 at 
      room temperature)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>15 mM 
      MgCl<SUB>2</SUB></FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Autoclave the 10x buffer 
      for 10 minutes at 15 psi (1.05 kg/cm <SUP>2</SUP>) on liquid cycle. Divide 
      the sterile buffer into aliquots and store them at 
  -20°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>KCl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Dissolve an appropriate 
      amount of solid KCl in H<SUB>2</SUB>O, autoclave for 20 minutes on liquid 
      cycle and store at room temperature. Ideally, this 4 M solution should be 
      divided into small (approx. 100 µl) aliquots in sterile tubes and each 
      aliquot thereafter used one time.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Tris-Cl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve 121.1 g of Tris base in 800 ml of H<SUB>2</SUB>O. Adjust the 
      pH to the desired value by adding concentrated HCl.</P>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2><I>pH</I></FONT> </TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2><I>HCl</I></FONT> 
        </TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.4 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>70 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.6 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>60 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>8.0 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>42 ml 
        </FONT></TD></TR></TBODY></TABLE>
      <P>(1 M) Allow the solution to cool to room temperature before making 
      final adjustments to the pH. Adjust the volume of the solution to 1 liter 
      with H<SUB>2</SUB>O. Dispense into aliquots and sterilize by 
      autoclaving.</P>
      <P>If the 1 M solution has a yellow color, discard it and obtain Tris of 
      better quality. The pH of Tris solutions is temperature-dependent and 
      decreases approx. 0.03 pH units for each 1°C increase in temperature. For 
      example, a 0.05 M solution has pH values of 9.5, 8.9, and 8.6 at 5°C, 
      25°C, and 37°C, respectively.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>dNTP Solution</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve each dNTP (deoxyribonucleoside triphosphates) in 
      H<SUB>2</SUB>O at an approximate concentration of 100 mM. Use 0.05 M Tris 
      base and a micropipette to adjust the pH of each of the solutions to 7.0 
      (use pH paper to check the pH). Dilute an aliquot of the neutralized dNTP 
      appropriately, and read the optical density at the wavelengths given in 
      the table below. Calculate the actual concentration of each dNTP. Dilute 
      the solutions with H<SUB>2</SUB>O to a final concentration of 50 mM dNTP. 
      Store each separately at -70°C in small aliquots. For polymerase chain 
      reactions (PCRs), adjust the dNTP solution to pH 8.0 with 2 N NaOH. 
      Commercially available solutions of PCR-grade dNTPs require no 
      adjustment.</P>
      <P>
      <TABLE width=400>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>Base</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana 
            size=2>wavelength<BR>(nm)</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>Extinction 
            Coefficient<BR>(E)&nbsp;(M<SUP>-1</SUP>cm<SUP>-1</SUP>)</FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>A</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>259</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.54 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>G</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>253</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.37 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>C</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>271</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.10 x 
            10<SUP>3</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>T</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>267</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.60 x 
            10<SUP>3</SUP></FONT></TD></TR></TBODY></TABLE></P>
      <P>For a cuvette with a path length of 1 cm, absorbance = EM. 100 mM stock 
      solutions of each dNTP are commercially available 
    (Pharmacia).</P></FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;     helvetica" size=3><STRONG>CAUTIONS</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" width=11 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Chloroform</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Chloroform CHCl<SUB>3</SUB> is irritating to the skin, eyes, 
      mucous membranes, and respiratory tract. It is a carcinogen and may damage 
      the liver and kidneys. It is also volatile. Avoid breathing the vapors. 
      Wear appropriate gloves and safety glasses. Always use in a chemical fume 
      hood. </FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Phenol:chloroform</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Phenol:chloroform, see Phenol; Chloroform 
</FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>REFERENCES</STRONG> 
</FONT><BR>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Foley K.P., 
  Leonard M.W., and Engel J.D. 1993. Quantitation of RNA using the polymerase 
  chain reaction. <I>Trends Genet.</I> 9:380-385. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Jones M.D. 
  1993. Reverse transcription of mRNA by <I>Thermus aquaticus</I> DNA polymerase 
  followed by polymerase chain reaction amplification. <I>Methods Enzymol</I> 
  218:413-419. </FONT></LI></OL><BR><BR><BR><BR><BR><FONT 
face=Arial,Verdana,Helvetica size=1><STRONG>Copyright © 2000 by Cold Spring 
Harbor Laboratory Press. All rights reserved.</STRONG><BR>No part of these 
pages, either text or image may be used for any purpose other than personal use. 
Therefore, reproduction modification, storage in a retrieval system or 
retransmission, in any form or by any means, electronic, mechanical, or 
otherwise, for reasons other than personal use, is strictly prohibited without 
prior written permission.</FONT><BR><BR>
</td></tr></table>
</BODY></HTML>
