<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Molecular Cloning | Protocol 13 Print Version</TITLE>
<META http-equiv=Content-Type content="text/html; charset=windows-1251"><!--================== Designed and Developed by WriteDesign =================--><!--======================= Powered by RomeBlack, Inc. =======================--><!--=== Copyright 2000, 2001 Cold Spring Harbor Labs. All Rights Reserved. ===-->
<META content="" name=keywords>
<META 
content="The following protocol can be used to amplify DNA up to 25 kb in length. To reduce the chance of contamination with exogenous DNAs, prepare and use a special set of reagents and solutions for PCR only. Bake all glassware for 6 hours at 150°C and autoclave all plasticware." 
name=description>
<META content="MSHTML 5.50.4134.600" name=GENERATOR></HEAD>
<BODY bgColor=#ffffff background=../izo/fon.gif>
<table width=600 border="0" cellspacing="0" cellpadding="0"><tr><td width=600><IMG height=49 hspace=4 src="../izo/logo_csh.gif" 
width=169 align=left vspace=4><IMG height=49 hspace=4 
src="../izo/print_header.gif" width=293 align=left vspace=4><BR 
clear=all><BR><FONT face="arial,verdana, helvetica" size=4><STRONG>Chapter 8, 
Protocol 13</STRONG></FONT><BR><BR clear=all><FONT face=arial,verdana,helvetica 
size=3><STRONG>Long PCR</STRONG></FONT><BR><BR><FONT 
face="arial,verdana, helvetica" size=2>The following protocol can be used to 
amplify DNA up to 25 kb in length. To reduce the chance of contamination with 
exogenous DNAs, prepare and use a special set of reagents and solutions for PCR 
only. Bake all glassware for 6 hours at 150°C and autoclave all 
plasticware.</FONT> <BR><BR>
<TABLE border=1>
  <TBODY>
  <TR>
    <TD width=700><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" 
      width=11 border=0><FONT face=arial,verdana,helvetica 
      size=2><STRONG>CAUTION</STRONG></FONT><BR><IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0><FONT 
      face="arial,verdana, helvetica" size=2> <STRONG>RECIPE</STRONG></FONT> 
  </TD></TR></TBODY></TABLE><BR><BR><FONT face=arial,verdana,helvetica 
size=3><STRONG>MATERIALS</STRONG></FONT><BR><BR>
<TABLE cellPadding=0 cellborder="0">
  <TBODY>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Buffers and Solutions</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Chloroform 
      </FONT><IMG height=11 alt=caution hspace=4 src="../izo/triangle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>dNTP solution 
      (20 mM) containing all four dNTPs (pH 8.0)</FONT> <IMG height=12 alt="" 
      hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>10x Long PCR 
      buffer</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Enzymes and Buffers</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Thermostable 
      DNA polymerase mix</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Klentaq1 
      can be obtained from AB Peptides (St. Louis, Missouri), and </I>Pfu<I> 
      polymerase can be obtained from Stratagene. A typical mixture contains 
      0.187 unit of </I>Pfu<I> and 33.7 units of Klentaq1 in a total volume of 
      1.2 µl.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Nucleic Acids and Oligonucleotides</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Forward 
      primers (20 µM) in H<SUB>2</SUB>O</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Reverse 
      primers (20 µM) in H<SUB>2</SUB>O (20 pmoles/µl)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Template 
      DNA</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Long PCR 
      works well on a variety of templates including recombinant PACs, BACs, 
      cosmids, and bacteriophage <IMG src="../izo/lambda.gif" border=0> 
      clones, as well as high-molecular-weight genomic DNAs. However, the 
      quality of the DNA is paramount. The average length of the template DNAs 
      (assayed by agarose gel or pulsed-field gel electrophoresis) should be at 
      least three times greater than the length of the desired PCR product. The 
      DNAs should also be extensively purified by equilibrium density 
      centrifugation in CsCl gradients, followed by dialysis against TE (pH 8.0) 
      (please see <A href="../01/10.htm">Chapter 
      1, Protocol 10</A>).</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Additional Reagents</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 4 of this 
      protocol requires the reagents listed in <A href="../05/02.htm">Chapter 
      5, Protocol 2</A> and <A href="../06/10.htm">Chapter 
      6, Protocol 10</A>.</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 
  border=0></TD></TR></TBODY></TABLE><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" 
size=3><STRONG>METHOD</STRONG></FONT> <BR><FONT face="arial,verdana, helvetica" 
color=#000000 size=2>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>In a 
  thin-walled amplification tube, add and mix in the following order: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10x long PCR 
        buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>20 mM solution 
        of four dNTPs</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>20 mM forward 
        primer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>20 mM reverse 
        primer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>thermostable 
        DNA polymerase mix</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>0.2 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>template 
        DNA</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>100 pg to 2 
        µg</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>to 50 
      µl</FONT></TD></TR></TBODY></TABLE><I>Templates purified from individual 
  recombinant clones constructed in bacteriophage <IMG src="../izo/lambda.gif" 
  border=0>, cosmid, bacteriophage P1, PAC, and BAC vectors should be used in 
  amounts ranging from 100 pg to 300 ng. Larger amounts of total genomic DNAs 
  are required, usually between 100 ng and 1 µg per reaction. The optimum amount 
  of template and the optimum ratio of primers:template should be ascertained 
  empirically for each new preparation of DNA.</I><BR><I>Primers used for long 
  PCR are generally slightly longer (25-30 nucleotides) than those used for 
  standard PCR. It is particularly important to strive for equality between the 
  melting temperatures of the two primers. If the difference in melting 
  temperatures exceeds one centigrade degree, mispriming and preferential 
  amplification of one strand may become a problem.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>If the thermal 
  cycler is not fitted with a heated lid, overlay the reaction mixtures with 1 
  drop (approx. 50 µl) of light mineral oil. Alternatively, place a bead of 
  paraffin wax into the tube if using a hot start PCR. Place the tubes or the 
  microtiter plate in the thermal cycler. Amplify the nucleic acids using the 
  denaturation, annealing, and polymerization times and temperatures listed in 
  the table. 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2><B>Cycle 
        Number</B></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2><B>Denaturation</B></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2><B>Annealing</B></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2><B>Polymerization</B></FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>24 
      cycles</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 min at 
        94°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 min at 
        60-67°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5-20 min at 
        68°C</FONT></TD></TR></TBODY></TABLE><I>Times and temperatures may need to be 
  adapted to suit the particular reaction conditions.</I><BR><I>The temperature 
  used for the annealing step depends on the melting temperature of the 
  oligonucleotide primers. Because the primers used in long PCR are generally 
  27-30 nucleotides in length, the annealing temperatures used in long PCR can 
  be considerably higher than those used in standard PCR.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>If mineral oil 
  was used to overlay the reaction (Step 2), remove the oil from the sample by 
  extraction with 150 µl of chloroform.<BR><I>The aqueous phase, which contains 
  the amplified DNA, will form a micelle near the meniscus. The micelle can be 
  transferred to a fresh tube with an automatic 
  micropipette.</I><BR><B>IMPORTANT</B> <I>Do not attempt chloroform extractions 
  in microtiter plates. The plastic used in these plates is not resistant to 
  organic solvents.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Analyze an 
  aliquot of the aqueous phase by electrophoresis through an agarose gel using 
  markers of an appropriate size. In many cases, the amount of amplified product 
  may be too small to be detected by conventional staining with ethidium 
  bromide. In this case, stain the DNA in the gel with SYBR Gold or transfer to 
  a nylon or nitrocellulose filter and probe by Southern hybridization (please 
  see <A href="../06/10.htm">Chapter 
  6, Protocol 10</A>). </FONT></LI></OL></FONT><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>RECIPES</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/circle.gif" width=12 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Ammonium sulfate 
      (NH<SUB>4</SUB>)<SUB>2</SUB>SO<SUB>4</SUB></STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare a 1 M 
      solution, dissolve 1.3 g of solid 
      (NH<SUB>4</SUB>)<SUB>2</SUB>SO<SUB>4</SUB> in a final volume of 10 ml of 
      H<SUB>2</SUB>O. Filter-sterilize the solution and store the solution at 
      room temperature.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Long PCR buffer</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>500 mM Tris-Cl (pH 9.0 at 
      room temperature)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>160 mM ammonium 
      sulfate</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>25 mM 
      MgCl<SUB>2</SUB></FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>1.5 mg/ml bovine serum 
      albumin</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Sterilize the 10x buffer by filtration through an 0.22-µm membrane. 
      Divide the sterile buffer into aliquots and store the aliquots at 
      -20°C.</P>
      <P>KCl (100 mM) can be used in place of ammonium sulfate in the 10x long 
      PCR buffer, and gelatin can be used at a concentration of 0.01% in the 
      final reaction in place of bovine serum albumin.</P>
      <P>Additional components to improve the efficiency of long PCR include 
      <I>glycerol</I> at a concentration of 5% (v/v) in the final reaction 
      mixture to promote separation of DNA strands at lower temperatures, and 
      <I>EDTA</I> at a concentration of 0.75 mM in the final reaction mixture to 
      chelate divalent cations such as Mn<SUP>2+</SUP> that might promote 
      scission of DNA strands.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Tris-Cl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve 121.1 g of Tris base in 800 ml of H<SUB>2</SUB>O. Adjust the 
      pH to the desired value by adding concentrated HCl.</P>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2><I>pH</I></FONT> </TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2><I>HCl</I></FONT> 
        </TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.4 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>70 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.6 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>60 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>8.0 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>42 ml 
        </FONT></TD></TR></TBODY></TABLE>
      <P>(1 M) Allow the solution to cool to room temperature before making 
      final adjustments to the pH. Adjust the volume of the solution to 1 liter 
      with H<SUB>2</SUB>O. Dispense into aliquots and sterilize by 
      autoclaving.</P>
      <P>If the 1 M solution has a yellow color, discard it and obtain Tris of 
      better quality. The pH of Tris solutions is temperature-dependent and 
      decreases approx. 0.03 pH units for each 1°C increase in temperature. For 
      example, a 0.05 M solution has pH values of 9.5, 8.9, and 8.6 at 5°C, 
      25°C, and 37°C, respectively.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>dNTP Solution</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve each dNTP (deoxyribonucleoside triphosphates) in 
      H<SUB>2</SUB>O at an approximate concentration of 100 mM. Use 0.05 M Tris 
      base and a micropipette to adjust the pH of each of the solutions to 7.0 
      (use pH paper to check the pH). Dilute an aliquot of the neutralized dNTP 
      appropriately, and read the optical density at the wavelengths given in 
      the table below. Calculate the actual concentration of each dNTP. Dilute 
      the solutions with H<SUB>2</SUB>O to a final concentration of 50 mM dNTP. 
      Store each separately at -70°C in small aliquots. For polymerase chain 
      reactions (PCRs), adjust the dNTP solution to pH 8.0 with 2 N NaOH. 
      Commercially available solutions of PCR-grade dNTPs require no 
      adjustment.</P>
      <P>
      <TABLE width=400>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>Base</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana 
            size=2>wavelength<BR>(nm)</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>Extinction 
            Coefficient<BR>(E)&nbsp;(M<SUP>-1</SUP>cm<SUP>-1</SUP>)</FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>A</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>259</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.54 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>G</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>253</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.37 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>C</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>271</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.10 x 
            10<SUP>3</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>T</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>267</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.60 x 
            10<SUP>3</SUP></FONT></TD></TR></TBODY></TABLE></P>
      <P>For a cuvette with a path length of 1 cm, absorbance = EM. 100 mM stock 
      solutions of each dNTP are commercially available 
    (Pharmacia).</P></FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;     helvetica" size=3><STRONG>CAUTIONS</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" width=11 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Chloroform</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Chloroform CHCl<SUB>3</SUB> is irritating to the skin, eyes, 
      mucous membranes, and respiratory tract. It is a carcinogen and may damage 
      the liver and kidneys. It is also volatile. Avoid breathing the vapors. 
      Wear appropriate gloves and safety glasses. Always use in a chemical fume 
      hood. </FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>REFERENCES</STRONG> 
</FONT><BR>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Barnes W.M. 
  1994. PCR-amplification of up to 35-kb DNA with high fidelity and high yield 
  from lambda bacteriophage templates. <I>Proc. Natl. Acad. Sci.</I> 
  91:2216-2220. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Foord O.S. and 
  Rose E.A. 1994. Long-distance PCR. <I>PCR Methods Appl.</I> 3:S149-161. 
  </FONT></LI></OL><BR><BR><BR><BR><BR><FONT face=Arial,Verdana,Helvetica 
size=1><STRONG>Copyright © 2000 by Cold Spring Harbor Laboratory Press. All 
rights reserved.</STRONG><BR>No part of these pages, either text or image may be 
used for any purpose other than personal use. Therefore, reproduction 
modification, storage in a retrieval system or retransmission, in any form or by 
any means, electronic, mechanical, or otherwise, for reasons other than personal 
use, is strictly prohibited without prior written 
permission.</FONT><BR><BR>
</td></tr></table>
</BODY></HTML>
