<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Molecular Cloning | Protocol 15 Print Version</TITLE>
<META http-equiv=Content-Type content="text/html; charset=windows-1251"><!--================== Designed and Developed by WriteDesign =================--><!--======================= Powered by RomeBlack, Inc. =======================--><!--=== Copyright 2000, 2001 Cold Spring Harbor Labs. All Rights Reserved. ===-->
<META content="" name=keywords>
<META 
content="Quantitative PCR involves co-amplification of two templates: a constant amount of a preparation containing the desired target sequence and serial dilutions of a reference template that is added in known amounts to a series of amplification reactions. The concentration of the target sequence is determined by simple interpolation into a standard curve. To reduce the chance of contamination with exogenous DNAs, prepare and use a special set of reagents and solutions for PCR only. Bake all glassware for 6 hours at 150°C and autoclave all plasticware." 
name=description>
<META content="MSHTML 5.50.4134.600" name=GENERATOR></HEAD>
<BODY bgColor=#ffffff background=../izo/fon.gif>
<table width=600 border="0" cellspacing="0" cellpadding="0"><tr><td width=600><IMG height=49 hspace=4 src="../izo/logo_csh.gif" 
width=169 align=left vspace=4><IMG height=49 hspace=4 
src="../izo/print_header.gif" width=293 align=left vspace=4><BR 
clear=all><BR><FONT face="arial,verdana, helvetica" size=4><STRONG>Chapter 8, 
Protocol 15</STRONG></FONT><BR><BR clear=all><FONT face=arial,verdana,helvetica 
size=3><STRONG>Quantitative PCR</STRONG></FONT><BR><BR><FONT 
face="arial,verdana, helvetica" size=2>Quantitative PCR involves 
co-amplification of two templates: a constant amount of a preparation containing 
the desired target sequence and serial dilutions of a reference template that is 
added in known amounts to a series of amplification reactions. The concentration 
of the target sequence is determined by simple interpolation into a standard 
curve. To reduce the chance of contamination with exogenous DNAs, prepare and 
use a special set of reagents and solutions for PCR only. Bake all glassware for 
6 hours at 150°C and autoclave all plasticware.</FONT> <BR><BR>
<TABLE border=1>
  <TBODY>
  <TR>
    <TD width=700><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" 
      width=11 border=0><FONT face=arial,verdana,helvetica 
      size=2><STRONG>CAUTION</STRONG></FONT><BR><IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0><FONT 
      face="arial,verdana, helvetica" size=2> <STRONG>RECIPE</STRONG></FONT> 
  </TD></TR></TBODY></TABLE><BR><BR><FONT face=arial,verdana,helvetica 
size=3><STRONG>MATERIALS</STRONG></FONT><BR><BR>
<TABLE cellPadding=0 cellborder="0">
  <TBODY>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Buffers and Solutions</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>10x 
      Amplification buffer</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>dNTP solution 
      (20 mM) containing all four dNTPs (pH 8.0)</FONT> <IMG height=12 alt="" 
      hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>MgCl<SUB>2</SUB> (50 mM)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Placental 
      RNase inhibitor (20 units/µl)</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Enzymes and Buffers</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Appropriate 
      restriction endonuclease</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Please see 
      Step 7.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Reverse 
      transcriptase (100-200 units/µl)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Thermostable 
      DNA polymerase. </FONT></TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Taq<I> DNA 
      polymerase is the standard and appropriate enzyme for the amplification 
      stage of most forms of quantitative PCR.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Radioactive Compounds</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>[<IMG 
      src="../izo/alpha.gif" border=0>-<SUP>32</SUP>P]dCTP (sp. act. 3000 
      Ci/mmole) at 10 mCi/ml </FONT><IMG height=11 alt=caution hspace=4 
      src="../izo/triangle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Nucleic Acids and Oligonucleotides</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Externally 
      added reference </FONT></TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Use a DNA 
      or RNA reference of known concentration to measure the concentration of 
      DNA sequences by conventional PCR and, whenever possible, an RNA reference 
      to measure the concentration of a target RNA by quantitative RT-PCR (<A href="../15/02.htm">Chapter 
      15, Protocol 2</A>).</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Sense primer 
      (20 µM) in H<SUB>2</SUB>O</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Antisense 
      primer (20 µM) in H<SUB>2</SUB>O. </FONT></TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Each primer 
      should be 20-30 nucleotides in length and contain approximately equal 
      numbers of the four bases, with a balanced distribution of G and C 
      residues and a low propensity to form stable secondary 
      structures.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Target nucleic 
      acid </FONT></TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>The target 
      can be a preparation of DNA or RNA, either total or poly(A)<SUP>+</SUP>. 
      Dissolve preparations of total RNA at a concentration of 0.5-1 mg/ml and 
      preparations of poly(A)<SUP>+</SUP> RNA at a concentration of 10-100 
      µg/ml. Dissolve DNA targets in 10 mM Tris-Cl (pH 7.6) at the following 
      concentrations: mammalian genomic DNA, 100 µg/ml; yeast genomic DNA, 1 
      µg/ml; bacterial genomic DNA, 0.1 µg/ml; and plasmid DNA, 1-5 
      ng/ml.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 
  border=0></TD></TR></TBODY></TABLE><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" 
size=3><STRONG>METHOD</STRONG></FONT> <BR><FONT face="arial,verdana, helvetica" 
color=#000000 size=2>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Design and 
  prepare a reference template suitable for the task at hand. Measure the 
  concentration of the reference template as carefully as possible, preferably 
  by fluorometry. Alternatively, estimate the amount of reference template after 
  gel electrophoresis and ethidium bromide staining. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Make a series 
  of tenfold dilutions (in H<SUB>2</SUB>O) containing concentrations of the 
  reference template ranging from 10<SUP>-6</SUP> to 10<SUP>-12</SUP> M. After 
  using the dilutions in Step 3, they should be stored at -70°C for use in Step 
  8. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>If starting 
  from RNA, denature the target RNA by incubating aliquots for 5 minutes at 
  75°C, followed by rapid chilling in ice water. Then, without delay, set up a 
  series of reverse transcription reactions containing increasing amounts of 
  reference template in sterile 0.5-ml microfuge tubes. For each reaction in the 
  series, prepare the following: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10x 
        amplification buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>20 mM solution 
        of four dNTPs (pH 8.0)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>20 µM antisense 
        primer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2.5 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>approx. 20 
        units/µl placental RNase inhibitor</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>50 mM 
        MgCl<SUB>2</SUB></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>denatured 
        target RNA</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10 pg to 1.0 
        µg</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>100-200 
        units/µl reverse transcriptase</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>tenfold 
        dilution of reference template</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>to 20 
      µl</FONT></TD></TR></TBODY></TABLE><I>MgCl<SUB>2</SUB> is added to meet the 
  needs of the reverse transcriptase.</I><BR>Incubate the reaction for 60 
  minutes at 37°C and then denature the reverse transcriptase by heating to 95°C 
  for 20 minutes. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>In sterile 
  0.5-ml microfuge tubes, amplification tubes, or the wells of a sterile 
  microtiter plate, set up amplification reactions with each reaction in the 
  series from Step 3: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>reverse 
        transcriptase reaction (Step 3) or target DNA</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>20 µM sense 
        primer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1.5 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>20 µM antisense 
        primer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1.25 
      µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10x 
        amplification buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>[<IMG 
        src="../izo/alpha.gif" border=0>-<SUP>32</SUP>P]dCTP (3000 
        Ci/mmole)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10 
    µCi</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>20 mM solution 
        of four dNTPs</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2><I>Taq</I> DNA 
        polymerase</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2 
      units</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>to 50 
      µl</FONT></TD></TR></TBODY></TABLE><B>IMPORTANT</B> <I>Do not reduce the 
  concentration of unlabeled dCTP in the reaction mixture to increase the 
  specific activity of the precursor pool. There is a danger that the amount of 
  the nucleotide could become limiting at late stages in the amplification 
  reaction.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>If the thermal 
  cycler is not fitted with a heated lid, overlay the reaction mixtures with 1 
  drop (approx. 50 µl) of light mineral oil. Alternatively, place a bead of wax 
  into the tube if using hot start PCR. Place the tubes or the microtiter plate 
  in the thermal cycler. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Amplify the 
  nucleic acids using the denaturation, annealing, and polymerization times and 
  temperatures listed in the table. 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2><B>Cycle 
        Number</B></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2><B>Denaturation</B></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2><B>Annealing</B></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2><B>Polymerization</B></FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>30 
      cycles</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>30 sec at 
        95°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>30 sec at 
        55°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 min at 
        72°C</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Last 
        cycle</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 min at 
        95°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>30 sec at 
        55°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 min at 
        72°C</FONT></TD></TR></TBODY></TABLE><I>Times and temperatures may need to be 
  adapted to suit the particular reaction conditions.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Analyze and 
  quantitate the amplified products.<BR><BR><STRONG>When using a reference 
  template that differs from the target sequence in size</STRONG> 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>a.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Analyze the 
        sizes of the amplified products in a 20-µl aliquot of each of the 
        reactions by gel electrophoresis and autoradiography.</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>b.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Excise the 
        amplified bands of the control template and target sequences from the 
        gel, and measure the amount of radioactivity in each band in a liquid 
        scintillation counter. Alternatively, scan the gel with the appropriate 
        detector (e.g., Ambis scanner or phosphorimager).</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>c.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Calculate the 
        relative amounts of the two radiolabeled DNAs in each of the 
      PCRs.</FONT></TD></TR></TBODY></TABLE><I>Correct the amount of radioactivity 
  to allow for differences in the molecular weights of the two radiolabeled 
  DNAs.</I><BR><BR><B>When using a reference template that contains a novel 
  restriction site or lacks a naturally occurring site</B> 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>a.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Heat the 
        samples to 94°C for 5 minutes following the final round of 
        amplification.</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>b.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Allow the 
        samples to cool gradually to room temperature and then digest a 20-µl 
        aliquot of each of the reactions with the appropriate restriction 
        enzyme.</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>c.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Analyze the 
        sizes of the amplified DNA fragments by gel electrophoresis and 
        autoradiography.</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>d.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Excise the 
        amplified bands of the control template and target sequences from the 
        gel, and measure the amount of radioactivity in each band in a liquid 
        scintillation counter. Alternatively, scan the gel with the appropriate 
        detector (e.g, Ambis scanner or phosphor-imager).</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>e.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Calculate the 
        relative amounts of the two radiolabeled DNAs in each of the 
      PCRs.</FONT></TD></TR></TBODY></TABLE><I>Correct the amount of radioactivity 
  to allow for differences in the molecular weights of the two radiolabeled 
  DNAs.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Examine the 
  results to determine the concentration of reference template that yields 
  approximately the same amount of amplified product as the target sequence. Set 
  up a second series of amplification reactions (please see Step 4) containing a 
  narrower range of concentrations of reference template.<BR><I>It is best to 
  generate this series of dilutions from the appropriate tenfold dilution of the 
  reference template (Step 2).</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Repeat Steps 5, 
  6, and 7. For each amplification reaction, measure the ratio of the yield of 
  amplified reference template to the yield of amplified target sequence. Plot 
  this ratio against the amount of reference template added to each 
  amplification reaction. From the resulting straight line, determine the 
  equivalence point (i.e., the amount of reference template that gives exactly 
  the same quantity of amplified product as the target sequence in the 
  reaction). Calculate the concentration of the target sequence in the original 
  sample. </FONT></LI></OL></FONT><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>RECIPES</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/circle.gif" width=12 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Amplification Buffer</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>500 mM KCl</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>100 mM Tris-Cl (pH 8.3 at 
      room temperature)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>15 mM 
      MgCl<SUB>2</SUB></FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Autoclave the 10x buffer 
      for 10 minutes at 15 psi (1.05 kg/cm <SUP>2</SUP>) on liquid cycle. Divide 
      the sterile buffer into aliquots and store them at 
  -20°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>KCl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Dissolve an appropriate 
      amount of solid KCl in H<SUB>2</SUB>O, autoclave for 20 minutes on liquid 
      cycle and store at room temperature. Ideally, this 4 M solution should be 
      divided into small (approx. 100 µl) aliquots in sterile tubes and each 
      aliquot thereafter used one time.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Tris-Cl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve 121.1 g of Tris base in 800 ml of H<SUB>2</SUB>O. Adjust the 
      pH to the desired value by adding concentrated HCl.</P>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2><I>pH</I></FONT> </TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2><I>HCl</I></FONT> 
        </TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.4 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>70 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.6 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>60 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>8.0 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>42 ml 
        </FONT></TD></TR></TBODY></TABLE>
      <P>(1 M) Allow the solution to cool to room temperature before making 
      final adjustments to the pH. Adjust the volume of the solution to 1 liter 
      with H<SUB>2</SUB>O. Dispense into aliquots and sterilize by 
      autoclaving.</P>
      <P>If the 1 M solution has a yellow color, discard it and obtain Tris of 
      better quality. The pH of Tris solutions is temperature-dependent and 
      decreases approx. 0.03 pH units for each 1°C increase in temperature. For 
      example, a 0.05 M solution has pH values of 9.5, 8.9, and 8.6 at 5°C, 
      25°C, and 37°C, respectively.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>dNTP Solution</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve each dNTP (deoxyribonucleoside triphosphates) in 
      H<SUB>2</SUB>O at an approximate concentration of 100 mM. Use 0.05 M Tris 
      base and a micropipette to adjust the pH of each of the solutions to 7.0 
      (use pH paper to check the pH). Dilute an aliquot of the neutralized dNTP 
      appropriately, and read the optical density at the wavelengths given in 
      the table below. Calculate the actual concentration of each dNTP. Dilute 
      the solutions with H<SUB>2</SUB>O to a final concentration of 50 mM dNTP. 
      Store each separately at -70°C in small aliquots. For polymerase chain 
      reactions (PCRs), adjust the dNTP solution to pH 8.0 with 2 N NaOH. 
      Commercially available solutions of PCR-grade dNTPs require no 
      adjustment.</P>
      <P>
      <TABLE width=400>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>Base</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana 
            size=2>wavelength<BR>(nm)</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>Extinction 
            Coefficient<BR>(E)&nbsp;(M<SUP>-1</SUP>cm<SUP>-1</SUP>)</FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>A</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>259</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.54 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>G</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>253</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.37 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>C</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>271</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.10 x 
            10<SUP>3</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>T</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>267</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.60 x 
            10<SUP>3</SUP></FONT></TD></TR></TBODY></TABLE></P>
      <P>For a cuvette with a path length of 1 cm, absorbance = EM. 100 mM stock 
      solutions of each dNTP are commercially available 
    (Pharmacia).</P></FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;     helvetica" size=3><STRONG>CAUTIONS</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" width=11 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Radioactive substances</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Radioactive substances: When planning an experiment that involves 
      the use of radioactivity, consider the physico-chemical properties of the 
      isotope (half-life, emission type, and energy), the chemical form of the 
      radioactivity, its radioactive concentration (specific activity), total 
      amount, and its chemical concentration. Order and use only as much as 
      needed. Always wear appropriate gloves, lab coat, and safety goggles when 
      handling radioactive material. X-rays and gamma rays are electromagnetic 
      waves of very short wavelengths either generated by technical devices or 
      emitted by radioactive materials. They might be emitted isotropically from 
      the source or may be focused into a beam. Their potential dangers depend 
      on the time period of exposure, the intensity experienced, and the 
      wavelengths used. Be aware that appropriate shielding is usually made of 
      lead or other similar material. The thickness of the shielding is 
      determined by the energy(s) of the X-rays or gamma rays. Consult the local 
      safety office for further guidance in the appropriate use and disposal of 
      radioactive materials. Always monitor thoroughly after using 
      radioisotopes. A convenient calculator to perform routine radioactivity 
      calculations can be found at: <A target=_blank 
      href="http://www.graphpad.com/calculators/radcalc.cfm">http://www.graphpad.com/calculators/radcalc.cfm</A>. 
      </FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>REFERENCES</STRONG> 
</FONT><BR>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Ferre F. 1992. 
  Quantitative or semi-quantitative PCR: Reality versus myth. <I>PCR Methods 
  Appl.</I> 2:1-9. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Freeman W.M., 
  Walker S.J., and Vrana K.E. 1999. Quantitative RT-PCR: Pitfalls and potential. 
  <I>BioTechniques</I> 26:112-125. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Halford W.P. 
  1999. The essential prerequisites for quantitative PCR. <I>Nat. 
  Biotechnol.</I> 17:835. </FONT></LI></OL><BR><BR><BR><BR><BR><FONT 
face=Arial,Verdana,Helvetica size=1><STRONG>Copyright © 2000 by Cold Spring 
Harbor Laboratory Press. All rights reserved.</STRONG><BR>No part of these 
pages, either text or image may be used for any purpose other than personal use. 
Therefore, reproduction modification, storage in a retrieval system or 
retransmission, in any form or by any means, electronic, mechanical, or 
otherwise, for reasons other than personal use, is strictly prohibited without 
prior written permission.</FONT><BR><BR>
</td></tr></table>
</BODY></HTML>
