<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Molecular Cloning | Protocol 9 Print Version</TITLE>
<META http-equiv=Content-Type content="text/html; charset=windows-1251"><!--================== Designed and Developed by WriteDesign =================--><!--======================= Powered by RomeBlack, Inc. =======================--><!--=== Copyright 2000, 2001 Cold Spring Harbor Labs. All Rights Reserved. ===-->
<META content="" name=keywords>
<META 
content="The double-stranded DNA of recombinant plasmid, phagemid, or bacteriophage M13 replicative form DNA is digested with two restriction enzymes whose sites of cleavage both lie between one end of the target DNA and the binding site for universal primer. The enzyme that cleaves nearer the target sequence must generate either a blunt end or a recessed 3&acute; terminus; the other enzyme must generate a four-nucleotide protruding 3&acute; terminus. Because only the blunt or recessed 3&acute; terminus of the resulting linear DNA is susceptible to exonuclease III, digestion proceeds unidirectionally away from the site of cleavage and into the target DNA. The exposed single strands are then removed by digestion with nuclease S1 or mung bean nuclease, and the DNA is then recircularized. If desired, a synthetic linker can be inserted at the site of recircularization." 
name=description>
<META content="MSHTML 5.50.4134.600" name=GENERATOR></HEAD>
<BODY bgColor=#ffffff background=../izo/fon.gif>
<table width=600 border="0" cellspacing="0" cellpadding="0"><tr><td width=600><IMG height=49 hspace=4 src="../izo/logo_csh.gif" 
width=169 align=left vspace=4><IMG height=49 hspace=4 
src="../izo/print_header.gif" width=293 align=left vspace=4><BR 
clear=all><BR><FONT face="arial,verdana, helvetica" size=4><STRONG>Chapter 13, 
Protocol 9</STRONG></FONT><BR><BR clear=all><FONT face=arial,verdana,helvetica 
size=3><STRONG>Generation of Sets of Nested Deletion Mutants with Exonuclease 
III</STRONG></FONT><BR><BR><FONT face="arial,verdana, helvetica" size=2>The 
double-stranded DNA of recombinant plasmid, phagemid, or bacteriophage M13 
replicative form DNA is digested with two restriction enzymes whose sites of 
cleavage both lie between one end of the target DNA and the binding site for 
universal primer. The enzyme that cleaves nearer the target sequence must 
generate either a blunt end or a recessed 3&acute; terminus; the other enzyme must 
generate a four-nucleotide protruding 3&acute; terminus. Because only the blunt or 
recessed 3&acute; terminus of the resulting linear DNA is susceptible to exonuclease 
III, digestion proceeds unidirectionally away from the site of cleavage and into 
the target DNA. The exposed single strands are then removed by digestion with 
nuclease S1 or mung bean nuclease, and the DNA is then recircularized. If 
desired, a synthetic linker can be inserted at the site of 
recircularization.</FONT> <BR><BR>
<TABLE border=1>
  <TBODY>
  <TR>
    <TD width=700><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" 
      width=11 border=0><FONT face=arial,verdana,helvetica 
      size=2><STRONG>CAUTION</STRONG></FONT><BR><IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0><FONT 
      face="arial,verdana, helvetica" size=2> <STRONG>RECIPE</STRONG></FONT> 
  </TD></TR></TBODY></TABLE><BR><BR><FONT face=arial,verdana,helvetica 
size=3><STRONG>MATERIALS</STRONG></FONT><BR><BR>
<TABLE cellPadding=0 cellborder="0">
  <TBODY>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Buffers and Solutions</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>dNTP solution 
      containing dATP, dGTP, and dTTP, each at 0.5 mM in 25 mM Tris-Cl (pH 
      8.0)</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Ethanol (100%, 
      ice cold, and 70%, room temperature)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>10x 
      Exonuclease III buffer</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Nuclease S1 
      stop mixture (13-9)</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>Phenol:chloroform</FONT> <IMG height=11 alt=caution hspace=4 
      src="../izo/triangle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Sodium acetate 
      (3 M, pH 5.2)</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Enzymes and Buffers</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Exonuclease 
      III</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>The quality 
      of exonuclease III varies from manufacturer to manufacturer and should be 
      checked in analytical digests before large-scale preparation of nested 
      deletion templates.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><BR>Klenow 
      mixture (sufficient for 30 samples)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana 
          size=2>H<SUB>2</SUB>O</FONT></TD>
          <TD width=5></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>20 µl</FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>1 M 
            MgCl<SUB>2</SUB></FONT></TD>
          <TD width=5></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>6 µl</FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>0.1 M Tris-Cl (pH 
            7.6)</FONT></TD>
          <TD width=5></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>3 µl</FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>Klenow 
          fragment</FONT></TD>
          <TD width=5></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>3 
        units</FONT></TD></TR></TBODY></TABLE></I><I>Prepare this mixture on ice 
      just before use.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><BR>Ligase 
      mixture (sufficient for 24 samples)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana 
          size=2>H<SUB>2</SUB>O</FONT></TD>
          <TD width=5></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>550 µl</FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>10x bacteriophage T4 
            ligation buffer</FONT></TD>
          <TD width=5></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>100 µl</FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>5 mM rATP</FONT></TD>
          <TD width=5></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>100 µl</FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>PEG 8000 (30% 
            w/v)</FONT></TD>
          <TD width=5></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>250 µl</FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>bacteriophage T4 DNA 
            ligase</FONT></TD>
          <TD width=5></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>5 Weiss 
        units</FONT></TD></TR></TBODY></TABLE></I><I>Prepare this mixture on ice 
      just before use.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><BR>Nuclease 
      S1 reaction mixture</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana 
          size=2>H<SUB>2</SUB>O</FONT></TD>
          <TD width=5></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>172 µl</FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>10x S1 
buffer</FONT></TD>
          <TD width=5></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>27 µl</FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>Nuclease S1</FONT></TD>
          <TD width=5></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>60 
        units</FONT></TD></TR></TBODY></TABLE></I><I>Prepare this mixture just 
      before use.</I><BR><I>Mung bean nuclease can be substituted for nuclease 
      S1 in this reaction mixture.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Restriction 
      enzymes (two)</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Nucleic Acids and Oligonucleotides</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Target 
      DNA</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Analyze the 
      sequence of the target DNA for the presence of suitable restriction sites. 
      Clone the DNA to be digested into a plasmid or bacteriophage vector that 
      contains as few nonessential sequences as possible. Analyze an aliquot of 
      the preparation of recombinant plasmid DNA that is to be used as a 
      substrate for mutagenesis by agarose gel electrophoresis in TAE buffer 
      (please see <A href="../05/01.htm">Chapter 
      5, Protocol 1</A>). The plasmid must be &gt;90% superhelical molecules. 
      Repurify the preparation if any linear or &gt;10% nicked or relaxed 
      plasmids are detected.</I><BR><I>Because exonuclease III will initiate 
      digestion from single-strand nicks, it is important that the template DNA 
      consist predominantly of closed circular molecules. Purification of the 
      template has the added advantage of removing small pieces of DNA and RNA 
      from the closed circular DNA preparation. These can interfere with 
      digestion by exonuclease III.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Additional Reagents</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 14 of 
      this protocol requires the reagents listed in <A href="../01/24.htm">Chapter 
      1, Protocol 24</A>, <A href="../01/25.htm">Chapter 
      1,Protocol 25</A>, or <A href="../01/26.htm">Chapter 
      1, Protocol 26</A> (for transformation).</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 15 of 
      this protocol requires the reagents listed in <A href="../01/01.htm">Chapter 
      1, Protocol 1</A> or <A href="../01/04.htm">Chapter 
      1, Protocol 4</A> (for minipreparation of plasmid DNA) or in <A href="../03/03.htm">Chapter 
      3, Protocol 3</A> (for preparation of replicative form of M13 DNA).</FONT> 
    </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 16 of 
      this protocol requires the reagents listed in <A href="../12/03.htm">Chapter 
      12, Protocol 3</A>, <A href="../12/04.htm">Chapter 
      12, Protocol 4</A>, or <A href="../12/05.htm">Chapter 
      12, Protocol 5</A>.</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 
  border=0></TD></TR></TBODY></TABLE><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" 
size=3><STRONG>METHOD</STRONG></FONT> <BR><FONT face="arial,verdana, helvetica" 
color=#000000 size=2>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Digest 10 µg of 
  target DNA (recombinant bacteriophage M13 replicative form DNA, phagemid DNA, 
  or plasmid DNA) with two restriction enzymes that cleave the polycloning site 
  between the primer-binding site of the vector and the target DNA. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Purify the DNA 
  by standard extraction with phenol:chloroform and precipitation with ethanol. 
  Carefully remove the supernatant, and add 0.5 ml of 70% ethanol to the 
  pellet.<BR><I>Rinsing the pellet with ethanol is important because sodium ions 
  inhibit exonuclease III (Hoheisel 1993).</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Recover the 
  washed pellet of DNA by centrifuging at maximum speed for 2 minutes at 4°C in 
  a microfuge, and then carefully remove the supernatant. Incubate the open tube 
  on the bench to allow the last traces of ethanol to evaporate, and then 
  dissolve the DNA in 60 µl of 1x exonuclease III buffer. Store the dissolved 
  DNA on ice. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Place 7.5 µl of 
  nuclease S1 reaction mixture in each of 25 0.5-ml microfuge tubes or in 25 
  wells of a 96-well microtiter plate with U-shaped wells. Store the microtiter 
  plate or microfuge tubes on a bed of ice. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Incubate the 
  DNA solution prepared in Step 3 for 5 minutes at 37°C. Transfer 2.5 µl of the 
  solution to the first microfuge tube or well of the microtiter plate 
  containing the nuclease S1 reaction mixture. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>To the 
  remainder of the DNA solution, add 150 units of exonuclease III per pmole of 
  recessed 3&acute; termini (1 unit of exonuclease III will generate 1 nmole of 
  acid-soluble total nucleotide in 30 minutes at 37°C). Tap the tube to mix the 
  contents and immediately return the tube to the 37°C water bath. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>At 30-second 
  intervals, remove 2.5-µl samples of the DNA solution and place them in 
  successive microfuge tubes or wells of the microtiter plate containing the 
  nuclease S1 reaction mixture. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>When all of the 
  samples have been harvested, incubate the microfuge tubes or microtiter plate 
  containing the nuclease S1 and digested plasmid DNA for 30 minutes at 30°C. 
  </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Add 1 µl of 
  nuclease S1 stop mixture (13-9) to each of the microfuge tubes or wells and 
  incubate the reaction mixtures for 10 minutes at 70°C. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Transfer the 
  microfuge tubes or microtiter plate to a bed of ice and analyze aliquots of 
  each of the samples by agarose gel electrophoresis. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Pool the 
  samples containing DNA fragments of the desired size. Add 1 µl of Klenow 
  mixture for each 10 µl of pooled sample and incubate the reaction mixture for 
  5 minutes at 37°C. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>For each 10 µl 
  of pooled sample, add 1 µl of 0.5 mM dNTPs. Continue incubation for 15 minutes 
  at room temperature. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Add 40 µl of T4 
  bacteriophage ligase mixture for each 10 µl of pooled sample. Mix and continue 
  incubation for 2 hours at room temperature. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Transform the 
  appropriate <I>E. coli</I> host with aliquots of the ligated DNA. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Prepare 
  minipreparations of bacteriophage M13 replicative form DNA, plasmid, or 
  phagemid DNA from at least 24 randomly selected plaques or colonies. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Linearize the 
  DNAs by digestion with an appropriate restriction enzyme and analyze their 
  sizes by electrophoresis through a 1% agarose gel. Include the original 
  plasmid or bacteriophage DNA that has been linearized by restriction enzyme 
  digestion as a marker. Choose clones of an appropriate size for sequencing 
  (Chapter 12) or additional restriction enzyme mapping. 
</FONT></LI></OL></FONT><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>RECIPES</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/circle.gif" width=12 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>EDTA</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare EDTA at 0.5 M 
      (pH 8.0): Add 186.1 g of disodium EDTA2H<SUB>2</SUB>O to 800 ml of 
      H<SUB>2</SUB>O. Stir vigorously on a magnetic stirrer. Adjust the pH to 
      8.0 with NaOH (approx. 20 g of NaOH pellets). Dispense into aliquots and 
      sterilize by autoclaving. The disodium salt of EDTA will not go into 
      solution until the pH of the solution is adjusted to approx. 8.0 by the 
      addition of NaOH.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Exonuclease III buffer</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>660 mM Tris-Cl (pH 
      8.0)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>66 mM 
      MgCl<SUB>2</SUB></FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>100 mM <IMG 
      src="../izo/beta.gif" border=0>-mercaptoethanol</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Add <IMG 
      src="../izo/beta.gif" border=0>-mercaptoethanol to the 10x stock just 
      before use.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Nuclease S1 stop mixture 
      (13-9)</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>0.3 M Tris 
  base</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>50 mM EDTA (pH 
    8.0)</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Sodium Acetate</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare a 3 M 
      solution: Dissolve 408.3 g of sodium acetate3H<SUB>2</SUB>O in 800 ml of 
      H<SUB>2</SUB>O. Adjust the pH to 5.2 with glacial acetic acid or adjust 
      the pH to 7.0 with dilute acetic acid. Adjust the volume to 1 liter with 
      H<SUB>2</SUB>O. Dispense into aliquots and sterilize by 
    autoclaving.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Tris-Cl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve 121.1 g of Tris base in 800 ml of H<SUB>2</SUB>O. Adjust the 
      pH to the desired value by adding concentrated HCl.</P>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2><I>pH</I></FONT> </TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2><I>HCl</I></FONT> 
        </TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.4 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>70 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.6 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>60 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>8.0 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>42 ml 
        </FONT></TD></TR></TBODY></TABLE>
      <P>(1 M) Allow the solution to cool to room temperature before making 
      final adjustments to the pH. Adjust the volume of the solution to 1 liter 
      with H<SUB>2</SUB>O. Dispense into aliquots and sterilize by 
      autoclaving.</P>
      <P>If the 1 M solution has a yellow color, discard it and obtain Tris of 
      better quality. The pH of Tris solutions is temperature-dependent and 
      decreases approx. 0.03 pH units for each 1°C increase in temperature. For 
      example, a 0.05 M solution has pH values of 9.5, 8.9, and 8.6 at 5°C, 
      25°C, and 37°C, respectively.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>dNTP Solution</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve each dNTP (deoxyribonucleoside triphosphates) in 
      H<SUB>2</SUB>O at an approximate concentration of 100 mM. Use 0.05 M Tris 
      base and a micropipette to adjust the pH of each of the solutions to 7.0 
      (use pH paper to check the pH). Dilute an aliquot of the neutralized dNTP 
      appropriately, and read the optical density at the wavelengths given in 
      the table below. Calculate the actual concentration of each dNTP. Dilute 
      the solutions with H<SUB>2</SUB>O to a final concentration of 50 mM dNTP. 
      Store each separately at -70°C in small aliquots. For polymerase chain 
      reactions (PCRs), adjust the dNTP solution to pH 8.0 with 2 N NaOH. 
      Commercially available solutions of PCR-grade dNTPs require no 
      adjustment.</P>
      <P>
      <TABLE width=400>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>Base</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana 
            size=2>wavelength<BR>(nm)</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>Extinction 
            Coefficient<BR>(E)&nbsp;(M<SUP>-1</SUP>cm<SUP>-1</SUP>)</FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>A</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>259</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.54 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>G</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>253</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.37 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>C</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>271</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.10 x 
            10<SUP>3</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>T</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>267</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.60 x 
            10<SUP>3</SUP></FONT></TD></TR></TBODY></TABLE></P>
      <P>For a cuvette with a path length of 1 cm, absorbance = EM. 100 mM stock 
      solutions of each dNTP are commercially available 
    (Pharmacia).</P></FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;     helvetica" size=3><STRONG>CAUTIONS</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" width=11 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>&szlig;-Mercaptoethanol </STRONG></FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">&szlig;-Mercaptoethanol (2-Mercaptoethanol) 
      HOCH<SUB>2</SUB>CH<SUB>2</SUB>SH may be fatal if inhaled or absorbed 
      through the skin and is harmful if ingested. High concentrations are 
      extremely destructive to the mucous membranes, upper respiratory tract, 
      skin, and eyes. &szlig;-Mercaptoethanol has a very foul odor. Wear appropriate 
      gloves and safety glasses. Always use in a chemical fume hood. 
  </FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Phenol:chloroform</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Phenol:chloroform, see Phenol; Chloroform 
</FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>REFERENCES</STRONG> 
</FONT><BR>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Henikoff S. 
  1984. Unidirectional digestion with exonuclease III creates targeted 
  breakpoints for DNA sequencing. <I>Gene</I> 28:351-359. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Henikoff S. 
  1987. Unidirectional digestion with exonuclease III in DNA sequence analysis. 
  <I>Methods Enzymol.</I> 155:156-165. </FONT></LI></OL><BR><BR><BR><BR><BR><FONT 
face=Arial,Verdana,Helvetica size=1><STRONG>Copyright © 2000 by Cold Spring 
Harbor Laboratory Press. All rights reserved.</STRONG><BR>No part of these 
pages, either text or image may be used for any purpose other than personal use. 
Therefore, reproduction modification, storage in a retrieval system or 
retransmission, in any form or by any means, electronic, mechanical, or 
otherwise, for reasons other than personal use, is strictly prohibited without 
prior written permission.</FONT><BR><BR>
</td></tr></table>
</BODY></HTML>
