<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Molecular Cloning | Protocol 1 Print Version</TITLE>
<META http-equiv=Content-Type content="text/html; charset=windows-1251"><!--================== Designed and Developed by WriteDesign =================--><!--======================= Powered by RomeBlack, Inc. =======================--><!--=== Copyright 2000, 2001 Cold Spring Harbor Labs. All Rights Reserved. ===-->
<META content="" name=keywords>
<META 
content="This traditional method is based on the Gubler-Hoffman method (Gubler and Hoffman 1983) and is divided into six stages:&#10;          &#10;            •Stage 1: Synthesis of First-strand cDNA Catalyzed by Reverse Transcriptase&#10;            •Stage 2: Second-strand Synthesis&#10;            •Stage 3: Methylation of cDNA&#10;            •Stage 4: Attachment of Linkers or Adaptors&#10;            •Stage 5: Fractionation of cDNA by Gel Filtration through Sepharose CL-4B&#10;            •Stage 6: Ligation of cDNA to Bacteriophage &#955; Arms&#10;          &#10;        This protocol describes the conversion of poly(A)+ mRNA into first-strand cDNA in a reaction catalyzed by a murine RNase H- reverse transcriptase and primed by oligo(dT), random hexamers, or prime-adapters. The reaction conditions for first-strand synthesis using other enzymes are given in Table 11-3 on p. 11.38 in the print version of the manual." 
name=description>
<META content="MSHTML 5.50.4134.600" name=GENERATOR></HEAD>
<BODY bgColor=#ffffff background=../izo/fon.gif>
<table width=600 border="0" cellspacing="0" cellpadding="0"><tr><td width=600><IMG height=49 hspace=4 src="../izo/logo_csh.gif" 
width=169 align=left vspace=4><IMG height=49 hspace=4 
src="../izo/print_header.gif" width=293 align=left vspace=4><BR 
clear=all><BR><FONT face="arial,verdana, helvetica" size=4><STRONG>Chapter 11, 
Protocol 1</STRONG></FONT><BR><BR clear=all><FONT face=arial,verdana,helvetica 
size=3><STRONG>Construction of cDNA Libraries<BR>Stage 1: Synthesis of 
First-strand cDNA Catalyzed by Reverse 
Transcriptase</STRONG></FONT><BR><BR><FONT face="arial,verdana, helvetica" 
size=2>This traditional method is based on the Gubler-Hoffman method (Gubler and 
Hoffman 1983) and is divided into six stages: 
<TABLE>
  <TBODY>
  <TR>
    <TD><FONT face=arial,helvetica,verdana size=2>•</FONT></TD>
    <TD><FONT face=arial,helvetica,verdana size=2>Stage 1: Synthesis of 
      First-strand cDNA Catalyzed by Reverse Transcriptase</FONT></TD></TR>
  <TR>
    <TD><FONT face=arial,helvetica,verdana size=2>•</FONT></TD>
    <TD><FONT face=arial,helvetica,verdana size=2>Stage 2: Second-strand 
      Synthesis</FONT></TD></TR>
  <TR>
    <TD><FONT face=arial,helvetica,verdana size=2>•</FONT></TD>
    <TD><FONT face=arial,helvetica,verdana size=2>Stage 3: Methylation of 
      cDNA</FONT></TD></TR>
  <TR>
    <TD><FONT face=arial,helvetica,verdana size=2>•</FONT></TD>
    <TD><FONT face=arial,helvetica,verdana size=2>Stage 4: Attachment of 
      Linkers or Adaptors</FONT></TD></TR>
  <TR>
    <TD><FONT face=arial,helvetica,verdana size=2>•</FONT></TD>
    <TD><FONT face=arial,helvetica,verdana size=2>Stage 5: Fractionation of 
      cDNA by Gel Filtration through Sepharose CL-4B</FONT></TD></TR>
  <TR>
    <TD><FONT face=arial,helvetica,verdana size=2>•</FONT></TD>
    <TD><FONT face=arial,helvetica,verdana size=2>Stage 6: Ligation of cDNA to 
      Bacteriophage <IMG src="../izo/lambda.gif" border=0> 
  Arms</FONT></TD></TR></TBODY></TABLE>This protocol describes the conversion of 
poly(A)<SUP>+</SUP> mRNA into first-strand cDNA in a reaction catalyzed by a 
murine RNase H<SUP>-</SUP> reverse transcriptase and primed by oligo(dT), random 
hexamers, or prime-adapters. The reaction conditions for first-strand synthesis 
using other enzymes are given in Table 11-3 on p. 11.38 in the print version of 
the manual.</FONT> <BR><BR>
<TABLE border=1>
  <TBODY>
  <TR>
    <TD width=700><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" 
      width=11 border=0><FONT face=arial,verdana,helvetica 
      size=2><STRONG>CAUTION</STRONG></FONT><BR><IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0><FONT 
      face="arial,verdana, helvetica" size=2> <STRONG>RECIPE</STRONG></FONT> 
  </TD></TR></TBODY></TABLE><BR><BR><FONT face=arial,verdana,helvetica 
size=3><STRONG>MATERIALS</STRONG></FONT><BR><BR>
<TABLE cellPadding=0 cellborder="0">
  <TBODY>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Buffers and Solutions</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Actinomycin 
      D</FONT> <IMG height=11 alt=caution hspace=4 src="../izo/triangle.gif" 
      width=12 border=0><IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Actinomycin 
      D is needed only when a wild-type version of Mo-MLV RT is used that 
      possesses RNase H activity.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>DTT 
      (dithiothreitol) (1 M)</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>EDTA (0.5 M, 
      pH 8.0)</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>KCl (1 
      M)</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>MgCl<SUB>2</SUB> (1 M)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Tris-Cl (1 M, 
      pH 8.3 at room temperature)</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Enzymes and Buffers</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Reverse 
      transcriptase</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Mo-MLV RT 
      is temperature-sensitive and should be stored at -20°C until needed at the 
      end of Step 1.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>RNase 
      inhibitor (optional)</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Nucleic Acids and Oligonucleotides</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>dNTP solution 
      containing all four dNTPs, each at 5 mM</FONT> <IMG height=12 alt="" 
      hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>Oligonucleotide primers for cDNA synthesis (1 mg/ml)</FONT> <IMG 
      height=12 alt="" hspace=4 src="../izo/circle.gif" width=12 border=0> 
  </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Synthesis 
      of first-strand cDNA is generally primed by random hexamers, 
      oligo(dT)<SUB>12-18</SUB>, or a mixture of the two. For directional 
      cloning, primer-adaptors of the general structure 
      <SUP>5'</SUP>p(dX)-(dR)-(dT)<SUB>x</SUB>-OH<SUP>3'</SUP> are used, where X 
      = a clamp composed of four nucleotides (usually GAGA), R = a recognition 
      site for a restriction enzyme, and (dT)<SUB>x</SUB> = a homopolymeric run 
      of 15 dT residues used for directional cloning.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>Poly(A)<SUP>+</SUP> RNA for cDNA synthesis (1 mg/ml)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2><I>Approximately 5-10 µg of <NOBR>poly(A)<SUP>+</SUP></NOBR> RNA is 
      required to synthesize enough double-stranded cDNA to construct a large 
      library. (One confluent 90-mm plate of cultured mammalian cells yields 1-2 
      µg of <NOBR>poly(A)<SUP>+</SUP></NOBR> RNA.) The synthetic reactions will 
      still work if less template is available, but the losses of cDNA at each 
      stage in the protocol will be proportionately greater.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>The 
      integrity of the poly(A)<SUP>+</SUP> RNA to be used should ideally be 
      checked by (1) agarose gel electrophoresis and northern hybridization with 
      a control probe (please see <A href="../07/05.htm">Chapter 
      7, Protocol 5</A> and <A href="../07/08.htm">Chapter 
      7, Protocol 8</A>), (2) translation in a cell-free system followed by 
      SDS-polyacrylamide gel electrophoresis of the resulting polypeptides, and 
      (3) analysis of the size of the first strand of cDNA synthesized in a 
      pilot reaction.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Radioactive Compounds</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>[<IMG 
      src="../izo/alpha.gif" border=0>-<SUP>32</SUP>P]dCTP (10 mCi/ml, 400 
      Ci/mmole)</FONT> <IMG height=11 alt=caution hspace=4 
      src="../izo/triangle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>[<IMG 
      src="../izo/alpha.gif" border=0>-<SUP>32</SUP>P]dATP and [<IMG 
      src="../izo/alpha.gif" border=0>-<SUP>32</SUP>P]dTTP should not be used 
      as radioactive tracers to monitor synthesis of the first and second 
      strands of cDNA, since they may be incorporated preferentially into cDNA 
      derived from the poly(A)<SUP>+</SUP> tract at the 3' terminus of the mRNA. 
      Batches of [<IMG src="../izo/alpha.gif" border=0>-<SUP>32</SUP>P]dCTP 
      &gt;2 weeks old should not be used in this protocol.</I><BR><I>Thaw the 
      [<IMG src="../izo/alpha.gif" border=0>-<SUP>32</SUP>P]dCTP just before 
      starting the protocol. Store the thawed radiolabeled dCTP on ice until 
      needed in Step 2. Return the radiolabeled dCTP to the freezer immediately 
      after use.</I><BR><B>IMPORTANT</B><I> The Stratagene cDNA Synthesis Kit 
      recommends the use of any label other than dCTP with its protocol. This 
      kit includes 5-methyl dCTP in the dNTP mixes to protect the cDNA against 
      cleavage at internal sites by </I>Xho<I>I, which is used to release 
      </I>Xho<I>I ends for ligation. The use of radiolabeled dCTP with the 
      Stratagene protocol could create unprotected </I>Xho<I>I sites within the 
      cDNA sequence.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Additional Reagents</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 5 of this 
      protocol requires the reagents listed in <A href="../05/08.htm">Chapter 
      5, Protocol 8</A>.</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 
  border=0></TD></TR></TBODY></TABLE><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" 
size=3><STRONG>METHOD</STRONG></FONT> <BR><FONT face="arial,verdana, helvetica" 
color=#000000 size=2>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>To synthesize 
  first-strand cDNA, mix the following in a sterile microfuge tube on ice: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 µg/µl 
        poly(A)<SUP>+</SUP> RNA</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 µg/µl 
        oligonucleotide primer(s)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 M Tris-Cl (pH 
        8.0 at 37°C)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2.5 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 M 
      KCl</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>3.5 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>250 mM 
        MgCl<SUB>2</SUB></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>solution of all 
        four dNTPs, each at 5 mM</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>0.1 M 
        dithiothreitol</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>RNase inhibitor 
        (optional)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>25 
      units</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>to 48 
      µl</FONT></TD></TR></TBODY></TABLE>Add the manufacturer's recommended amount 
  of Mo-MLV H<SUP>-</SUP> RT to the reaction. Mix the reagents well by gentle 
  vortexing.<BR><I>Mo-MLV RT is temperature-sensitive and should be stored at 
  -20°C until needed.</I><BR><B>IMPORTANT</B> <I>If a preparation of AMV RT is 
  used, please see Table 11-3 on p. 11.38 in the print version of the manual for 
  reaction conditions.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>After all of 
  the components of the reaction have been mixed at 0°C, transfer 2.5 µl of the 
  reaction to a fresh 0.5-ml microfuge tube. Add 0.1 µl of [<IMG 
  src="../izo/alpha.gif" border=0>-<SUP>32</SUP>P]dCTP (400 Ci/mmole, 10 
  mCi/ml) to the small-scale reaction. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Incubate the 
  large- and small-scale reactions for 1 hour at 37°C.<BR><I>Higher temperatures 
  (up to 55°C) can be used with some mutant forms of Mo-MLV RT.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>At the end of 
  the incubation period, add 1 µl of 0.25 M EDTA to the small-scale reaction 
  containing the radioisotope. Transfer the small-scale reaction to ice. Heat 
  the large-scale reaction to 70°C for 10 minutes and then transfer it to ice. 
  </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Measure both 
  the total amount of radioactivity and the amount of trichloroacetic acid 
  (TCA)-precipitable radioactivity in 0.5 µl of the small-scale reaction, as 
  described in Appendix 8 in the print version of the manual. In addition, it is 
  worthwhile analyzing the products of the small-scale reaction through an 
  alkaline agarose gel using appropriate DNA markers (please see <A href="../05/08.htm">Chapter 
  5, Protocol 8</A>).<BR><I>The remainder of the small-scale reaction can be 
  stored at -20°C.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Calculate the 
  amount of first-strand cDNA synthesized as follows: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>a.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Since 10 µl of 
        a solution containing all four dNTPs at a concentration of 5 mM each was 
        used (i.e., 10 µl of 20 mmoles/liter of total dNTP), the large-scale 
        reaction must contain<BR>20 nmoles/µl dNTP x 10 µl = 200 nmoles of 
        dNTP</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>b.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Because the 
        molecular weight of each dNMP incorporated into DNA is approx. 330 
        g/mole, the reaction is capable of generating a total of<BR>200 nmoles x 
        330 ng/nmole = 66 µg of DNA</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>c.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Therefore, from 
        the results of the small-scale reaction,<BR><U>cpm incorporated</U> x 66 
        µg = µg of first strand of cDNA 
        synthesized<BR>&nbsp;&nbsp;&nbsp;&nbsp;total 
  cpm</FONT></TD></TR></TBODY></TABLE></FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Proceed as soon 
  as it is feasible to the next stage in the synthesis of the cDNA. 
</FONT></LI></OL></FONT><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>RECIPES</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/circle.gif" width=12 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Actinomycin D</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Dissolve actinomycin D in 
      methanol at a concentration of 5 mg/ml. Store the stock solution at -20°C 
      in the dark.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>DTT</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare a 1 M 
      solution: Dissolve 3.09 g of dithiothreitol in 20 ml of 0.01 M sodium 
      acetate (pH 5.2) and sterilize by filtration. Dispense into 1-ml aliquots 
      and store at -20°C. Under these conditions, dithiothreitol is stable to 
      oxidation by air.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>EDTA</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare EDTA at 0.5 M 
      (pH 8.0): Add 186.1 g of disodium EDTA•2H<SUB>2</SUB>O to 800 ml of 
      H<SUB>2</SUB>O. Stir vigorously on a magnetic stirrer. Adjust the pH to 
      8.0 with NaOH (approx. 20 g of NaOH pellets). Dispense into aliquots and 
      sterilize by autoclaving. The disodium salt of EDTA will not go into 
      solution until the pH of the solution is adjusted to approx. 8.0 by the 
      addition of NaOH.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>KCl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Dissolve an appropriate 
      amount of solid KCl in H<SUB>2</SUB>O, autoclave for 20 minutes on liquid 
      cycle and store at room temperature. Ideally, this 4 M solution should be 
      divided into small (approx. 100 µl) aliquots in sterile tubes and each 
      aliquot thereafter used one time.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Oligonucleotide primers for cDNA 
      synthesis</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Synthesis of first-strand cDNA is generally primed by random hexamers, 
      oligo(dT)<SUB>12-18</SUB>, or a mixture of the two. For directional 
      cloning, primer-adaptors of the general structure 
      <SUP>5'</SUP>p(dX)-(dR)-(dT)<SUB>x</SUB>-OH<SUP>3'</SUP> are used, where X 
      = a clamp composed of four nucleotides (usually GAGA), R = a recognition 
      site for a restriction enzyme, and (dT)<SUB>x</SUB> = a homopolymeric run 
      of 15 dT residues used for directional cloning.</P>
      <P>Oligo(dT)<SUB>12-18</SUB> primers are present in the first-strand 
      reaction mixture at a 10-fold (Mo-MLV H<SUP>-</SUP>) to 40-fold (Mo-MLV) 
      molar excess. Unlike random hexamers and primer-adaptors (please see 
      below), the concentration of oligo(dT)<SUB>12-18</SUB> primers generally 
      does not need to be optimized in the reaction mixture.</P>
      <P>The concentration of random hexamers in the reaction mixture is 
      critical. As the ratio of hexamers to mRNA increases, the average length 
      of the cDNA decreases. It is therefore essential to optimize the ratio of 
      random primers and mRNA template in a series of pilot reactions containing 
      radiolabeled dCTP. The average size of the first-strand cDNA generated in 
      each of the pilot reactions should be measured by alkaline agarose gel 
      electrophoresis and autoradiography using DNA markers of known size. When 
      setting up a large-scale reaction to synthesize first-strand cDNA, use the 
      highest ratio of random primer to mRNA that generates cDNA of the maximum 
      average length.</P>
      <P>Primer-adaptors are generally less troublesome than random primers. 
      However, it is nevertheless prudent to optimize the ratio of 
      primer-adaptors and mRNA template in a series of pilot reactions 
      containing radiolabeled dCTP. Measure the amount of cDNA synthesized in 
      each reaction as outlined in Steps 5 and 6 of this protocol. When setting 
      up a large-scale reaction to synthesize first-strand cDNA, use the lowest 
      ratio of primer-adaptor to mRNA that generates the maximum yield of 
      cDNA.</P>
      <P>Prepare stock solutions of primers (1 mg/ml in 10 mM Tris-Cl [pH 7.4]). 
      Store aliquots of the solutions at 
-70°C.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Tris-Cl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve 121.1 g of Tris base in 800 ml of H<SUB>2</SUB>O. Adjust the 
      pH to the desired value by adding concentrated HCl.</P>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2><I>pH</I></FONT> </TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2><I>HCl</I></FONT> 
        </TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.4 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>70 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.6 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>60 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>8.0 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>42 ml 
        </FONT></TD></TR></TBODY></TABLE>
      <P>(1 M) Allow the solution to cool to room temperature before making 
      final adjustments to the pH. Adjust the volume of the solution to 1 liter 
      with H<SUB>2</SUB>O. Dispense into aliquots and sterilize by 
      autoclaving.</P>
      <P>If the 1 M solution has a yellow color, discard it and obtain Tris of 
      better quality. The pH of Tris solutions is temperature-dependent and 
      decreases approx. 0.03 pH units for each 1°C increase in temperature. For 
      example, a 0.05 M solution has pH values of 9.5, 8.9, and 8.6 at 5°C, 
      25°C, and 37°C, respectively.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>dNTP Solution</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve each dNTP (deoxyribonucleoside triphosphates) in 
      H<SUB>2</SUB>O at an approximate concentration of 100 mM. Use 0.05 M Tris 
      base and a micropipette to adjust the pH of each of the solutions to 7.0 
      (use pH paper to check the pH). Dilute an aliquot of the neutralized dNTP 
      appropriately, and read the optical density at the wavelengths given in 
      the table below. Calculate the actual concentration of each dNTP. Dilute 
      the solutions with H<SUB>2</SUB>O to a final concentration of 50 mM dNTP. 
      Store each separately at -70°C in small aliquots. For polymerase chain 
      reactions (PCRs), adjust the dNTP solution to pH 8.0 with 2 N NaOH. 
      Commercially available solutions of PCR-grade dNTPs require no 
      adjustment.</P>
      <P>
      <TABLE width=400>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>Base</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana 
            size=2>wavelength<BR>(nm)</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>Extinction 
            Coefficient<BR>(E)&nbsp;(M<SUP>-1</SUP>cm<SUP>-1</SUP>)</FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>A</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>259</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.54 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>G</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>253</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.37 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>C</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>271</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.10 x 
            10<SUP>3</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>T</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>267</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.60 x 
            10<SUP>3</SUP></FONT></TD></TR></TBODY></TABLE></P>
      <P>For a cuvette with a path length of 1 cm, absorbance = EM. 100 mM stock 
      solutions of each dNTP are commercially available 
    (Pharmacia).</P></FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;     helvetica" size=3><STRONG>CAUTIONS</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" width=11 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Actinomycin D</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Actinomycin D is a teratogen and a carcinogen. It is highly toxic 
      and may be fatal if inhaled, ingested, or absorbed through the skin. It 
      may also cause irritation. Avoid breathing the dust. Wear appropriate 
      gloves and safety glasses. Always use in a chemical fume hood. Solutions 
      of actinomycin D are light-sensitive. </FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Radioactive substances</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Radioactive substances: When planning an experiment that involves 
      the use of radioactivity, consider the physico-chemical properties of the 
      isotope (half-life, emission type, and energy), the chemical form of the 
      radioactivity, its radioactive concentration (specific activity), total 
      amount, and its chemical concentration. Order and use only as much as 
      needed. Always wear appropriate gloves, lab coat, and safety goggles when 
      handling radioactive material. X-rays and gamma rays are electromagnetic 
      waves of very short wavelengths either generated by technical devices or 
      emitted by radioactive materials. They might be emitted isotropically from 
      the source or may be focused into a beam. Their potential dangers depend 
      on the time period of exposure, the intensity experienced, and the 
      wavelengths used. Be aware that appropriate shielding is usually made of 
      lead or other similar material. The thickness of the shielding is 
      determined by the energy(s) of the X-rays or gamma rays. Consult the local 
      safety office for further guidance in the appropriate use and disposal of 
      radioactive materials. Always monitor thoroughly after using 
      radioisotopes. A convenient calculator to perform routine radioactivity 
      calculations can be found at: <A target=_blank 
      href="http://www.graphpad.com/calculators/radcalc.cfm">http://www.graphpad.com/calculators/radcalc.cfm</A>. 
      </FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>REFERENCES</STRONG> 
</FONT><BR>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Gerard G.F. and 
  D'Alessio J.M. 1993. Reverse transcriptase (EC2.7.7.49): The use of cloned 
  Moloney murine leukemia virus reverse transcriptase to synthesize DNA from 
  RNA. <I>Methods Mol. Biol.</I> 16:73-94. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Gubler U. and 
  Hoffman B.J. 1983. A simple and very efficient method for generating cDNA 
  libraries. <I>Gene</I> 25:263-269. </FONT></LI></OL><BR><BR><BR><BR><BR><FONT 
face=Arial,Verdana,Helvetica size=1><STRONG>Copyright © 2000 by Cold Spring 
Harbor Laboratory Press. All rights reserved.</STRONG><BR>No part of these 
pages, either text or image may be used for any purpose other than personal use. 
Therefore, reproduction modification, storage in a retrieval system or 
retransmission, in any form or by any means, electronic, mechanical, or 
otherwise, for reasons other than personal use, is strictly prohibited without 
prior written permission.</FONT><BR><BR>
</td></tr></table>
</BODY></HTML>
